%% acronyms
\newacronym{RNA}{RNA}{\textbf{R}ibo\textbf{n}ucleic \textbf{A}cid}
\newacronym{DNA}{DNA}{\textbf{D}eoxyribo\textbf{n}ucleic \textbf{A}cid}
\newacronym{PCR}{PCR}{\textbf{P}olymerase \textbf{C}hain \textbf{R}eaction}
\newacronym{EBI}{EBI}{\textbf{E}uropean \textbf{B}ioinformatics \textbf{I}nstitute}
%\newacronym{mRNA}{\textit{m}RNA}{\textit{\textbf{m}essenger}
%\textbf{R}ib\textbf{n}ucleic \textbf{a}cid}
\newacronym{GEO}{GEO}{\textbf{G}ene \textbf{E}xpression \textbf{O}mnibus}

\newacronym{HCD}{HCD}{\textbf{H}igher-energy \textbf{C}ollisional
\textbf{D}issociation}

\newacronym{CID}{CID}{\textbf{C}ollision-\textbf{I}nduced \textbf{D}issociation}

\newabbreviation[description={\textit{m}essenger \glsxtrshort{RNA}}]
{mRNA}{mRNA}{messenger RNA}

\newacronym{laser}{laser}{light amplification by stimulated emission of radiation}

\newacronym{SVM}{SVM}{Support Vector Machine}

\newacronym{RPKM}{RPKM}{\textbf{R}eads \textbf{P}er \textbf{K}ilobase of
transcript per \textbf{M}illion mapped \textbf{R}eads}

\newacronym{FPKM}{FPKM}{\textbf{F}ragments \textbf{P}er \textbf{K}ilobase of
transcript per \textbf{M}illion mapped \textbf{F}ragments}

\newacronym{FDR}{FDR}{\textbf{F}alse \textbf{d}iscovery \textbf{r}ate}

\newacronym{PSM}{PSM}{\textbf{P}eptide \textbf{S}pectrum \textbf{M}atch}

\newacronym{PTM}{PTM}{\textbf{P}ost \textbf{T}ranslational \textbf{M}odification}

\newacronym{SAM}{SAM}{\textbf{S}equence \textbf{Alignment}/\textbf{M}ap}

\newacronym{EM}{EM}{\textbf{E}xpectation-\textbf{M}aximisation}


\newabbreviation%
[description={\textit{B}GZF-compressed
\glsxtrshort{SAM}}]{BAM}{BAM}{BGZF-compressed SAM}

%\newacronym{ncRNA}{\textit{nc}RNA}{\textbf{n}on \textbf{c}oding \textbf{R}ibo\textbf{n}ucleic \textbf{A}cid}

\newacronym{NIH}{NIH}{\textbf{N}ational \textbf{I}nstitutes of \textbf{H}ealth
(USA)}

\newacronym{FFPE}{FFPE}{\textbf{F}ormalin-\textbf{F}ixed
\textbf{P}araffin-\textbf{E}mbedded}

\newabbreviation%
[description={\textit{N}on \textit{c}oding \glsxtrshort{RNA}}]
{ncRNA}{ncRNA}{Non coding RNA}

\newabbreviation%
[description={\textit{L}ong \textit{N}on \textit{c}oding \glsxtrshort{RNA}}]
{lncRNA}{lncRNA}{Long non-coding RNA}

\newabbreviation%
[description={\textit{S}mall \textit{n}ucle\textbf{o}lar \glsxtrshort{RNA}}]
{snoRNA}{snoRNA}{Small nucleolar RNA}

\newabbreviation%
[description={\textit{S}mall \textit{Ca}jal body \glsxtrshort{RNA}}]
{scaRNA}{scaRNA}{Small Cajal body RNA}

\newabbreviation%
[description={\textit{S}mall \textit{n}uclear ribonucleic \glsxtrshort{RNA}}]
{snRNA}{snRNA}{Small nuclear ribonucleic RNA}

\newabbreviation%
[description={\textit{S}mall \textit{c}onditional \glsxtrshort{RNA}}]
{scRNA}{scRNA}{Small conditional RNA}

\newabbreviation%
[description={\textit{T}ransfer \glsxtrshort{RNA}}]
{tRNA}{tRNA}{Transfer RNA}

\newabbreviation%
[description={\textit{c}omplementary \glsxtrshort{DNA}}]
{cDNA}{cDNA}{complementary DNA}

\newabbreviation%
[description={micro\glsxtrshort{RNA}}]
{miRNA}{miRNA}{microRNA}

\newabbreviation%
[description={Ribosomal \glsxtrshort{RNA}}]
{rRNA}{rRNA}{Ribosomal RNA}

\newabbreviation%
[description={\textit{d}eoxynucleoside \textit{t}ri\textit{p}hosphate.
The N indicates that it can be any
nucleotide (usually either Adenosine (A), Cytosine (C), Guanine (G) or
Thymine (T))}]{dNTP}{dNTP}{deoxynucleoside triphosphate}

%%%% glossary entries
\newglossaryentry{RNA-Seq}{name={RNA-Seq},
    description={RNA sequencing, which can also be called whole transcriptome
    shotgun sequencing}}

\newglossaryentry{DNA-Seq}{name={DNA-Seq},
    description={DNA sequencing, which can also be called whole genome
    shotgun sequencing}}

\newglossaryentry{miRNA-Seq}{name={miRNA-Seq},
    description={miRNA sequencing, which can also be called miRNA shotgun
    sequencing}}


\newglossaryentry{HBB}{name={HBB},description={Hemoglobin Subunit Beta}}

\newglossaryentry{RT-qPCR}{name={RT-qPCR},
description={\textbf{R}everse-\textbf{T}ranscription
\textbf{q}uantitative real-time \textbf{P}olymerase \textbf{C}hain
\textbf{R}eaction is a molecular biology technique to quantify the amount of
\glsxtrlong{RNA} in a given cell or sample. Cycles of monitored replications are
used to robustly measure the gene expression. It is often considered
to be the most powerful and sensitive of the quantitative assay for
\glsxtrlong{RNA}. However, this method requires to know in advance which are the
genes of interest.}}

\newglossaryentry{ArrayExpress}{name=ArrayExpress,description={EBI archive of
Functional Genomics Data. It stores data from high-throughput functional
genomics experiments, and provides these data for reuse to the
research community.}}

\newglossaryentry{ENA}{name=ENA,description={The European Nucleotide Archive
provides a comprehensive record of the world's nucleotide sequencing information,
covering raw sequencing data, sequence assembly information and functional
annotation.}}

\newglossaryentry{dbGaP}{name=dbGaP,description={The database of Genotypes and
Phenotypes (dbGaP) was developed to archive and distribute the data and results
from studies that have investigated the interaction of genotype and phenotype in
Humans.}}

\newglossaryentry{Pride}{name=PRIDE,description={PRoteomics IDEntifications
(PRIDE) database is a centralized, standards compliant, public data repository
for proteomics data, including protein and peptide identifications,
post-translational modifications and supporting spectral evidence.
PRIDE is a core member in the ProteomeXchange (PX) consortium,
which provides a single point for submitting mass spectrometry based proteomics
data to public-domain repositories.
Datasets are submitted to PRIDE via ProteomeXchange and are handled by expert
biocurators.}}

\newglossaryentry{Proteomicsdb}{name=Proteomicsdb,description={ProteomicsDB is a
joint effort of the Technische Universität München (TUM) and SAP SE\@. It is
dedicated to expedite the identification of the human proteome and its use across
the scientific community.}}

\newglossaryentry{GTEx}{name=GTEx, description={The \textbf{G}enotype-\textbf{T}issue
\textbf{Ex}pression project establishes a resource database and associated tissue
bank for the scientific community to study the relationship between genetic
variation and gene expression in human tissues.}}

\newglossaryentry{nt}{name={nt},description={\textbf{N}ucleo\textbf{t}id;
common unit of length for single-stranded nucleic acids}}

\newglossaryentry{bp}{name={bp},description={\textbf{B}ase {P}air;
unit of length for double-stranded nucleic acids}}

\newglossaryentry{SNP}{name={SNP},description={Single Nucleotide Polymorphism}}

\newglossaryentry{eQTL}{name={eQTL},
  description={expression Quantitative Trait Locus},
  %first={\glsentrydesc{eQTL} (\glsentrytext{eQTL})},
  plural={eQTL},
  descriptionplural={expression Quantitative Trait Loci},
  %firstplural={\glsentrydescplural{LED} (\glsentryplural{})}
}


\newglossaryentry{GTF}{name={GTF},
    description={\textbf{G}ene \textbf{T}ransfer \textbf{F}ormat is a tab-delimeted
    file format based on the \glsxtrshort{GFF} format and hold information about
    gene structure. A main feature of this file is that it can be validatable
    which increases the data reliability.}}

\newglossaryentry{GFF}{name={GFF},
    description={\textbf{G}eneral \textbf{F}eature \textbf{F}ormat is a tab-delimited
    file format that records gene information or other features as \glsxtrshort{DNA},
\glsxtrshort{RNA} or protein sequences.}}

\newacronym{LC}{LC}{\textbf{L}iquid \textbf{C}hromatography}
\newacronym{MS}{MS}{\textbf{M}ass \textbf{S}pectrometry}
\newabbreviation[description={\glsxtrshort{LC} followed by tandem \glsxtrshort{MS}}]
{LC-MS/MS}{LC-MS/MS}{LC followed by tandem MS}

\newabbreviation[description={tandem \glsxtrshort{MS}}]
{MS/MS}{MS/MS}{tandem MS}

\newacronym{FTMS}{FTMS}{\textbf{F}ourier \textbf{t}ransform \textbf{M}ass
\textbf{s}pectrometer}


\newacronym{SDS}{SDS}{\textbf{S}odium \textbf{D}odecyl \textbf{S}ulfate}
\newacronym{PAGE}{PAGE}{\textbf{P}oly\textbf{a}crylamide \textbf{G}el
\textbf{E}lectrophoresis}
\newabbreviation[description={\glsxtrlong{SDS}-\glsxtrlong{PAGE}}]
{SDS-PAGE}{SDS-PAGE}{Sodium Dodecyl Sulphate-\gls{PAGE}}


\newabbreviation[description={\textbf{R}eversed-\textbf{P}hase
\glsxtrshort{LC}}]
{RPLC}{RPLC}{Reversed-phase LC}


\newglossaryentry{Phred}{name=Phred,description={A Phred quality score is a
measure of the quality of the identification of the nucleobases generated by
automated DNA sequencing.}}


\newglossaryentry{GRCh}{name=GRCh, description={\textbf{G}enome \textbf{R}eference
\textbf{C}onsortium \textbf{h}uman genome build. It is always followed by a
version number.}}

\newglossaryentry{Perl}{name={Perl}, description={Perl langage is
an \textit{interpreted} programming langage developed by Larry Wall and
first released in 1987 to easily handle textual information.}}

\newglossaryentry{Python}{name={Python}, description={Python langage is a
\textit{high-level} and \textit{interpreted} programming langage developed by
Guido van Rossum and first released in 1991. The main purpose of this langage is
to be multivalent while enhancing code readibility.}}

\newglossaryentry{cluster}{name={Computer cluster}, text={computer cluster},
description={Set of connected
computers that work together to improve performance over a single computer}}

\newglossaryentry{GENCODE}{name={GENCODE},description={Project that produces
high quality reference gene annotation for human and mouse genomes.}}

\newglossaryentry{FASTQ}{name={FASTQ},description={File Format recording for
each cluster read, a unique identifier, a nucleotide sequence and the call
accuracy for each base (\gls{Phred} score). Optionally, there can be more
information, \eg\ the spatial position of the cluster on the flow cell.}}

\newglossaryentry{flow}{name={Flow cell},text={flow cell},
description={Support of Illumina
sequencing. It enables the parallelisation of the sequencing of millions of
\gls{DNA} fragments together which are kept spatially separated in clusters.
It is a glass slide with lanes and each lane is coated with short nucleotide
sequences that are used to hybridise by complementarity adapters on the \gls{DNA}
that will be sequenced.}}


\newglossaryentry{fusionG}{name={Fusion gene}, text={fusion gene},
description={It is a gene that is
the product of the fusion of parts of two different genes.}}
