\chapter*{Introduction}\label{ch:intro}
\addcontentsline{toc}{chapter}{INTRODUCTION}

%\setlength{\epigraphwidth}{0.53\textwidth}
\setlength{\epigraphwidth}{0.62\textwidth}
%\epigraphhead[155]{%
\setlength{\epigraphrule}{0pt}
\epigraphhead[5]{%
    \epigraph{\emph{I am not accustomed to saying anything with certainty after
    only one or two observations.}}{Andreas Vesalius \mycite{omalley_1964}}
}

%\vspace{2cm}

Nowadays, we have comprehensive knowledge
about the structure and functioning of the human body at the macroscopic level.
At the microscopic level, however,
the identification and mapping of the molecules, their function and whereabouts
still need to be refined --- or even, yet to be done.\mybr\

Beyond the invaluable addition to our knowledge,
other more practical reasons are also sustaining the effort
for human expression atlases.
Establishing subsets of genes with specific behaviour in particular conditions
can facilitate diagnosis or translate to effective drug targets.
Besides, a robust atlas for undiseased baseline expression will be
extremely useful for all the studies where controls are hard to sample,
which is generally the case in cancer research.

The completed and annotated human genome and
recent technological advances for high-throughput expression studies
have opened the way towards this future new milestone.
The recent explosion of high-throughput transcriptomic atlases in the literature
provides sufficient evidence of this.
Examples include mouse~\mycite{Wu2009-lw,Ringwald2012}, pig~\mycite{Freeman2012},
sheep~\mycite{Clark2017-mw}, plants as maize~\mycite{Stelpflug2016-sm},
vigna~\mycite{Yao2016-se}, pigeon pea~\mycite{Pazhamala2017-ig},
parasite, \eg\ \species{Schistosoma mansoni}.
Many focus on Human, as a whole,
\eg\ see~\mycite{Krupp2012,Jimenez-Lozano2012,Uhlen2015,GTEx2013}
or on specific aspects,
\eg\ the organogenesis in the embryos~\mycite{Gerrard2016-zu}.

There are high incentives to develop the atlases with
transcriptome shotgun sequencing (\ie\ \Rnaseq).
The technology involved in similar older projects\footnote{%
E.g.\ Gene Expression Atlas for Human Embryogenesis~\mycite{Yi2010-az},
Atlas of human primary cells~\mycite{Mabbott2013-xf},
Gene atlas of mouse and human protein-encoding transcriptomes
(now hosted by \soft{BioGPS}) \mycite{Su2004-kc},
Allen Brain Atlas~\mycite{Hawrylycz2012-mx}.}
has demonstrated to generate highly variable data~\mycite{Rung2013-ul}
that is challenging to integrate
(even when produced by the same laboratory
on the same platform)~\mycite{Walsh2015-nf}.
When I started my doctorate,
little was known about the interstudy robustness of \Rnaseq.
However, it had already shown reduced background noise and
a more extensive dynamic range of detection~\mycite{rnaseq-2009}.
Besides, \Rnaseq\ has the added advantage to discover new transcripts
as it does not rely on previous knowledge~\mycite{rnaseq-2009}.

Considering the growing number of studies referring these atlases\footnote{%
More than 2,800 papers for five Human primary studies on 15 September 2019.
}
or the numerous efforts to compile them in new resources for the community ---
\eg\
\hFoCi{TISSUES}{https://tissues.jensenlab.org/Search}{Santos2015-rj},
\hFoCi{Harmonizome}{https://amp.pharm.mssm.edu/Harmonizome/}{Harmonizome}
and, after reprocessing the raw data,
\hFoCi{Expression Atlas}{https://www.ebi.ac.uk/gxa/home}{EBIgxa}
---
assessing the consistency of the results from one study to another
has become paramount.
The recently reported reproducibility crisis in science \mycite{Begley2012-hm,%
Fatovich2017-lo,Lindner2018-qy,Lyu2018-ze}
has only further underlined this need.

Given the previous context,
the first aim of my doctorate was
to appraise the consistency of findings highlighted
in individual large scale transcriptomics of (undiseased) human tissues.
Then, with the publication of the first drafts of the
human proteome~\mycite{PandeyData,KusterData},
my aims have expanded to the integration of human tissue expression data
across different datasets and biological layers.

\section*{Outlook of this thesis}

In \Cref{ch:datasets},
I first present the five transcriptomic (\Rnaseq)
and the three proteomic (\glsxtrshort{MS}) datasets
I have preselected for my analysis.
Then I describe the bioinformatic pipelines
that have automated the processing of this massive amount of data.

After considering several biases sources and strategies to limit them
in \Cref{ch:expression},
I compare and integrate the independent transcriptomic datasets
in \Cref{ch:Transcriptomics}.
Following an assessment of the findings consistency in proteomics
in \Cref{ch:proteomics},
I then employ different approaches for integrating
transcriptomics and proteomics in \Cref{ch:Integration}.
Finally, I close this thesis with a few remarks in \Cref{ch:conclusion}.

However, before anything else,
I review the scientific background of my doctorate works
in the following \Cref{ch:background}.

