\chapter{Introduction}
\label{sec:Introduction}
\epigraph{All life is nucleic acid; the rest is commentary.}{I. Asimov -
"The Relativity of Wrong" (1988) - "Beginning with Bone" (May 1987)}


\Rough{
Gene expression, transcription, translation, regulation at each stage,
expected and reported correlation between transcripts and proteins
Measuring transcript expression by RNAseq, wet-lab part, analysis methods,
available large scale datasets on human tissues
Measuring protein expression by MS, data analysis – spectral counts, top 3, etc,
available large scale datasets on human tissues
The problem of comparing and integrating independent datasets, EBI’s GXA \\ \\
key concepts: DNA storage, RNA transfert of information and regulation,
protein effectors ==\textgreater phenotype
}

\begin{itemize}
    \item Proteomics messy and noisy, challenging \ldots
    \item Transcriptomics seem a good proxy to study the phenotype => one kind of molecule made of the same pieces.
    \item Technology improved a lot from the microarrays to sequencing.
\end{itemize}

\section{Life and expression}

\TK{The central dogma of Molecular biology}

\fixme{Add references on replications, not going into that one}

\subsection{Translation + Regulation}

\subsection{Translation + Regulation}

\TK{Assumption mRNA and proteins : highly correlated [find references]}


\section{Capturing the expression}
\subsection{Transcriptomics}
\subsection{Proteomics}

\section{Problem of integrating independent datasets ====> GXA}

\subsection{Consistency issues}

\subsection{Reproducibility issues : i.e. R upgrading, blablabla}
On fait un exemple stupide.



\begin{table}
    \begin{tabular}{@{}llr@{}} \toprule
        \multicolumn{2}{c}{Item} \\ \cmidrule(r){1-2}
        Animal & Description & Price (\$)\\ \midrule
        Gnat & per gram & 13.65 \\
        & each & 0.01 \\
        Gnu & stuffed & 92.50 \\
        Emu & stuffed & 33.33 \\
        Armadillo & frozen & 8.99 \\ \bottomrule
    \end{tabular}
\end{table}



et un vrai tableau: \ref{tabDesTrans5DF}

\begin{sidewaystable}{}
    \centering
    \caption{Technical description of the 5 transcriptomic dataset (\Rnaseq) used
    for this study}
    \label{tabDesTrans5DF}
    \begin{tabular}{@{}ccccccccc@{}}
        \toprule
        \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}ArrayExpress ID \\ or Consortium name\end{tabular}} & \multicolumn{2}{c}{\begin{tabular}[c]{@{}c@{}}Library\\ preparation\end{tabular}} & \multicolumn{2}{c}{Sequencing} & \multicolumn{2}{c}{Replicates} & \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Tissue\\ number\end{tabular}} & \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}Multi-sampling \\from the same individual\end{tabular}} \\ \cmidrule(lr){2-7}
            & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}Whole\\ RNA\end{tabular}} & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}PolyA\\ selected\end{tabular}} & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}Single\\end\end{tabular}} & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}Paire\\end\end{tabular}} &  \multicolumn{1}{c|}{Biological} & \multicolumn{1}{c|}{Technical} &  &  \\ \midrule
                E-MTAB-305 & Y &  & Y &  &  &  & 11 &  \\
                E-GEOD-303522 (VT) &  & Y & Y &  & Y &  & 8 &  \\
                \begin{tabular}[c]{@{}c@{}}E-MTAB-513,(IBM: Illumina\\ Body Map)\end{tabular} &  & Y & Y & Y &  & (Y) & 16 &  \\
                E-MTAB-28363 &  & Y &  & Y & Y & Y & 27 &  \\
                 GTEX (v1.4) 4 &  & Y &  & Y & Y &  & 54 & Y \\ \bottomrule
    \end{tabular}
\end{sidewaystable}

