\chapter*{Introduction}
\label{ch:intro}
\addcontentsline{toc}{chapter}{INTRODUCTION}

Over the past few years, a large number of gene expression atlases based on
\Rnaseq\ have been released for many different organisms\footnote{E.g.,
mouse~\mycite{Wu2009-lw,Ringwald2012}, pig~\mycite{Freeman2012},
sheep~\mycite{Clark2017-mw}, plants as maize~\mycite{Stelpflug2016-sm},
vigna~\mycite{Yao2016-se},pigeonpea~\mycite{Pazhamala2017-ig},
parasite, \eg\ \species{Schistosoma mansoni}.}.
Many focus on Human, as a whole,
\eg\ see~\mycite{Krupp2012,Jimenez-Lozano2012,Uhlen2015,GTEx2013}
or on on specific aspects,
\eg\ the organogenesis in the embryos~\mycite{Gerrard2016-zu}.

On the other hand, there is also a considerable sustained effort
to compile these studies in new ressources for the community;
either by relying on the initial expression levels released with the original
studies --- as for example
\hFoCi{TISSUES}{https://tissues.jensenlab.org/Search}{Santos2015-rj} or
\hFoCi{Harmonizome}{https://amp.pharm.mssm.edu/Harmonizome/}{Harmonizome}
--- or
by reprocessing the raw data as
\hFoCi{Expression Atlas}{https://www.ebi.ac.uk/gxa/home}{EBIgxa}.

While similar microarrays-based projects exist\footnote{E.g.\ Gene Expression
Atlas for Human Embryogenesis~\mycite{Yi2010-az}, Atlas of human primary
cells~\mycite{Mabbott2013-xf}, Gene atlas of mouse and human protein-encoding
transcriptomes (now hosted by \soft{BioGPS}) \mycite{Su2004-kc},
Allen Brain Atlas~\mycite{Hawrylycz2012-mx}.},
there is a burst in the use \Rnaseq\ as the main technology to establish
these references\footnote{Indeed, microarrays straightforwardly exhibit
a large drawback compared to \Rnaseq:
they allow to measure a finite number of already identified genes
while \Rnaseq\ has the capacity of discovering new ones.}.



The incentive is that compared to microarrays, there should be a lot less
batch effects and that these studies, in addition to explore the transcriptome
for new (undiscovered until now) transcripts, the relationships (coexpression,
coregulation between them) due to the fact that there are not these biases (as
microarrays) these studies can be used as reference for later studies.


Problem: while they don't present some of the batch effects of the microarrays,
there was no real studies \emph{at that time} on how good is the \Rnaseq.

While \emph{batch effects} were well-known for microarrays,
at the time I started this project
the subject was little known.
Since then several other studies have been published on the matter.
Whenever relevant, I present and discuss those results them relation to my owns.
The first piece towards this goal was done by the~\mycite{seqcmaqc}.

\begin{comment}
Normal tissues sampled from various people are phenotypically very similar,
despite individual hereditary and environmental variations.
At molecular level,
technical noise, batch effects and physical limitations
are also generating variations.
\end{comment}


Despite individual hereditary and environmental variations,
normal tissue samples are phenotypically very similar.
At the molecular level, in addition to the technical noise,
more variations are also expected.

Microarray studies have illustrated that integrating data
from different assays (even from the same platform and laboratory)
is challenging~\mycite{Walsh2015-nf},
which also explain the abundance of methods and tools~\mycite{Tseng2012-if}.
While meta-analyses or cross-normalisation of microarrays are still
used~\mycite{Afroz2016-xk} and they may be very pertinent and
useful,
microarray technology is subject to saturation effects and is semi-quantitative
while suffering from great batch effects~\mycite{Rung2013-ul}.

\Rnaseq\ nurtures the hope of enabling true quantitative measurements.


In \Paper{seqcmaqc}~\mycite{seqcmaqc} many facts that
we had observed within the lab have been reported.
Indeed, this paper highlights several consistent points
across sites and platforms,
such as:
\begin{itemize}[topsep=0pt,nolistsep,noitemsep]
        \setlength{\itemsep}{1pt}
        \setlength{\parskip}{0pt}
        \setlength{\parsep}{0pt}
    \item Detection of the same unannotated isoforms (provided a significant expression)
    \item Absolute quantification seems unachievable yet,
        but relative quantification is robust and even agrees well with \gls{qPCR}
        and microarrays.
    \item Comprehensive gene model crucial for accurate expression profiling.
    \item They advise to profile multiple libraries or samples per conditions.
    \item Gene-specific biases are observed for every platforms
        (including \gls{qPCR}).
    \item To achieve absolute quantification, it would need to estimate and
        remove background noise (improves accuracy but at the expense of precision)
    \item Pipelines have an impact on the final results.
\end{itemize}

Their results are sound as they have used the same samples across the different
sites and platforms which are very different biologically (Universal
Human Reference RNA) and Human Brain Reference RNA
(see~\mycite{MAQC_Consortium2006-hs}) (from the \gls{MAQC} consortium).


