\chapter{Introduction}
\label{ch:intro}



Over the past few years, a large number of gene expression atlases based on
\Rnaseq\ have been released for many different organisms\footnote{E.g.,
mouse~\mycite{Wu2009-lw,Ringwald2012}, pig~\mycite{Freeman2012},
sheep~\mycite{Clark2017-mw}, plants as maize~\mycite{Stelpflug2016-sm},
vigna~\mycite{Yao2016-se},pigeonpea~\mycite{Pazhamala2017-ig},
parasite, \eg\ \species{Schistosoma mansoni}.}.
Many focus on Human, as a whole,
\eg\ see~\mycite{Krupp2012,Jimenez-Lozano2012,Uhlen2015,GTEx2013}
or on on specific aspects,
\eg\ the organogenesis in the embryos~\mycite{Gerrard2016-zu}.

On the other hand, there is also a considerable sustained effort
to compile these studies in new ressources for the community;
either by relying on the initial expression levels released with the original
studies --- as for example
\hFoCi{TISSUES}{https://tissues.jensenlab.org/Search}{Santos2015-rj} or
\hFoCi{Harmonizome}{https://amp.pharm.mssm.edu/Harmonizome/}{Harmonizome}
--- or
by reprocessing the raw data as
\hFoCi{Expression Atlas}{https://www.ebi.ac.uk/gxa/home}{EBIgxa}.

While similar microarrays-based projects exist\footnote{E.g.\ Gene Expression
Atlas for Human Embryogenesis~\mycite{Yi2010-az}, Atlas of human primary
cells~\mycite{Mabbott2013-xf}, Gene atlas of mouse and human protein-encoding
transcriptomes (now hosted by \soft{BioGPS}) \mycite{Su2004-kc},
Allen Brain Atlas~\mycite{Hawrylycz2012-mx}.},
there is a burst in the use \Rnaseq\ as the main technology to establish
these references\footnote{Indeed, microarrays straightforwardly exhibit
a large drawback compared to \Rnaseq:
they allow to measure a finite number of already identified genes
while \Rnaseq\ has the capacity of discovering new ones.}.



The incentive is that compared to microarrays, there should be a lot less
batch effects and that these studies, in addition to explore the transcriptome
for new (undiscovered until now) transcripts, the relationships (coexpression,
coregulation between them) due to the fact that there are not these biases (as
microarrays) these studies can be used as reference for later studies.


Problem: while they don't present some of the batch effects of the microarrays,
there was no real studies \emph{at that time} on how good is the \Rnaseq.

While \emph{batch effects} were well-known for microarrays,
at the time I started this project
the subject was little known.
Since then several other studies have been published on the matter.
Whenever relevant, I present and discuss those results them relation to my owns.
The first piece towards this goal was done by the~\mycite{seqcmaqc}.
