\clearpage
\chapter*{Summary}
\addcontentsline{toc}{chapter}{Summary}
\label{ch:summary}

\begin{singlespace}

%    \textbf{Investigating the normal gene expression in Human:
%    from transcriptomic to proteomic.}
%
%    or

    \textbf{Expression in normal human tissues: learning from
    independent publicly available high-throughput transcriptomic and proteomic
    data}

    \begin{small}
    Over the past five years, many high-throughput expression studies focusing
    on normal human tissues have been published. While, several studied the
    transcriptomic expression with sequencing \gls{RNA} (\gls{RNA-Seq}), others
    studied the proteomic expression with Mass spectrometry. These recent assays
    are often used as individual resources to answer a demand of the community:
    are its observations either specific to a particular condition or disease or
    are they also done in normal conditions. Indeed, while we have seen
    an accumulation of primary cells/tissues, the normal samples within studies
    are usually very few, when they exist at all. Hence, we have observed the
    multiplication of expression atlases that try to provide
    a reference for normal conditions based on assays focusing on normal tissues.
    Although all of these assays present many overlaps on their comprising tissues,
    very few atlases try to integrate, in any way, their results together
    (either focusing on the transcriptomic or proteomic side or both sides
    together).

    First, I investigate the consistency of transcriptomic expression,
    in particular the expression of \mRNAs, between different
    published assays. I show that independent samples are more likely to
    cluster based on their biological origins than based on their original
    studies and that many genes present a consistent expression profile across
    tissues for many different and independent studies. Some of these genes could
    be described as house-keeping genes. I also show that we can reuse published
    data to assess roughly the biological quality of a sample. In the last
    results chapter, I present several attempts to integrate the gene expression
    profiles for a common set of tissues of different studies together.

    Then, after a brief exposition of the proteomic data currently available, I
    study the integration of transcriptomic with proteomic data. While recent
    studies combining proteomics and transcriptomics have failed to show good
    correlations between proteins and mRNAs, I show that when comparing the
    same tissues we observe good correlations despite the samples having
    different biological and technical backgrounds. Moreover, I found that there
    is a high similarity between tissue-specific mRNAs and tissues specific
    proteins and that other categories of mRNAs also present very high
    correlation of expression e.g.\ the metabolic genes.

    Overall, my thesis describes a global picture of the current consolidated
    knowledge we can extract from public data. It also provides new insights in
    the house-keeping and tissue-specific genes and proteins and, although we
    ought to be careful as I compared the results from different sources, the
    joint study of transcriptomic and proteomic highlihted genes whose
    translation is lesser regulated.

    \end{small}
\end{singlespace}
