\clearpage
\chapter*{Summary}
\addcontentsline{toc}{chapter}{Summary}
\label{ch:summary}

\begin{singlespace}

%    \textbf{Investigating the normal gene expression in Human:
%    from transcriptomic to proteomic.}
%
%    or

    \textbf{Expression in normal human tissues: learning from
    independent publicly available high-throughput transcriptomic and proteomic
    data}
    {\small Over the past five years, many high-throughput expression studies
    focusing on normal human tissues have been published. Several explore
    the gene expression with \gls{RNA} sequencing (\gls{RNA-Seq}) and others
    the proteomic expression with Mass Spectrometry. These recent assays are
    often used as individual resources to help the community to answer a
    recurrent question: are specific observations limited to a particular
    condition (or disease) or also commonly done in normal conditions.
    Indeed, the normal samples within studies are usually very few, when they
    exist at all. Hence, we have observed the multiplication of expression
    atlases that try to provide a reference for normal conditions based on
    these recent studies. Although all of these assays
    present many overlapping tissues, very few atlases try to integrate,
    in any way, their results together (either focusing only on the
    transcriptomic or proteomic side or both sides together).

    In the first results chapter, I investigate the consistency of
    transcriptomic expression, in particular the expression of \mRNAs,
    between different published assays. I show that independent samples are
    more likely to cluster based on their biological origins than based on
    their original studies and that many genes present a consistent expression
    profile across tissues for many different and independent studies. Some
    of these genes could be described as housekeeping genes. I show that we
    can reuse published data to assess roughly the biological quality of a
    sample. I also present several attempts to integrate together the gene
    expression values of a common set of tissues across different studies.

    After a brief exposition of the proteomic data currently available, I
    study the integration of transcriptomic with proteomic data. Despite
    recent studies combining proteomics and transcriptomics failing to show
    good correlations between proteins and \mRNAs, I show that when comparing
    the same tissues we observe good correlations despite the samples having
    different biological and technical backgrounds. Moreover, I find that
    there is a high similarity between tissue-specific \mRNAs\ and tissues
    specific proteins and that other categories of \mRNAs\ also present very
    high correlation of expression e.g.\ the metabolic genes.

    Overall, my thesis describes a global picture of the current consolidated
    knowledge we can extract from public data. It also provides new insights
    in the housekeeping and tissue-specific genes and proteins. Although we
    ought to be careful as I compared the results from different sources,
    this joint study of transcriptomic and proteomic also highlights genes
    whose translations are less regulated.
    } 
\end{singlespace}
