\RequirePackage[l2tabu,orthodox]{nag}
\documentclass[a4paper
,12pt
,headinclude
,footinclude
,BCOR=1cm
,parskip=half*
,open=right
%,listof=totoc
,floatperchapter
%,pdfspacing %doesn't work with xelatex
,numbers=noendperiod
,twoside=true
,headings=big
,bibliography=totoc
]{scrbook}
%{scrreprt}
\KOMAoptions{headsepline = true, footsepline = false}

%\usepackage{scrpage2}
\input{config.tex}
\input{glossary.tex}



\title{Expression in normal human tissues: learning from independent publicly
available high-throughput transcriptomic and proteomic data}
\subject{This dissertation is submitted for the degree of Doctor of Philosophy}
\author{Mitra Parissa Barzine}
\publishers{%
  \includegraphics[width=0.3\textwidth]{HughesHallShield}
  \\[0.5\baselineskip]
  Hughes Hall,\\
  University of Cambridge\\[0.2\baselineskip]
  EMBL-EBI}

\lowertitleback{%
  \centering
  {\footnotesize Licensed under Creative Commons Attribution International (CC BY) 4.0}
  \\[2mm]
  \href{http://creativecommons.org/licenses/by/4.0/}{%
    \raisebox{-0.25em}{\includegraphics[width=1em]{cc.pdf}
      \includegraphics[width=1em]{by.pdf}}}}

%\addtokomafont{section}{\Large}
%\addtokomafont{subsection}{\large}

\begin{document}
\hyphenation{%
  abun-dance
  be-twe-en
  chan-ges
  ri-bo-so-mal
  samp-ling
  ENS-EMBL
}

\pagenumbering{roman}
\setcounter{page}{1}
\maketitle

\pagestyle{scrheadings}
%\frontmatter\

\input{disclaimer.tex}
\input{summary.tex}
\input{acknowledgements.tex}

%% problem with page numbering fixed with:
%% http://tex.stackexchange.com/a/125090/102519
\begingroup
    %\hypersetup{linkcolor=thesis@toccolor}
    \cleardoublepage
    \pdfbookmark{\contentsname}{toc}
    \tableofcontents
    \phantomsection
    \printglossary[title=List of Terms and Abbreviations,toctitle=List of Terms and Abbreviations]\raggedbottom
    \phantomsection
    \listoffigures
    \addcontentsline{toc}{chapter}{\listfigurename}\raggedbottom\ \clearpage \thispagestyle{empty}
    \phantomsection
    \listoftables
    \addcontentsline{toc}{chapter}{\listtablename}\raggedbottom\ \clearpage \thispagestyle{empty}
    \cleardoublepage
\endgroup

%\input{listofdatasets.tex}
\cleardoublepage
%\mainmatter\
\pagenumbering{arabic}
\setcounter{page}{1}
\setcounter{table}{0}
\input{introduction.tex}
\input{datasets.tex}
\input{expressed.tex}
\input{transcriptomics.tex}
\input{proteomics.tex}
%\input{additional.tex}
\input{integration.tex}
\input{normalisation.tex}
\input{conclusion.tex}

%\backmatter\
\appendix\
%\input{appendix.tex}

\part*{\Huge\bfseries\addfontfeature{LetterSpace=10}APPENDIX}%\label{ch:appendix}
\addcontentsline{toc}{part}{APPENDIX}
\renewcommand\thechapter{\Alph{chapter}}
\begingroup

\input{app/fastq.tex}

\input{ds/castle.tex}
\section{Script to convert older format to FASTQ}% \texorpdfstring{\fastq}{FASTQ}}
\label{code:fastq}
Most tools only accept \fastq\ files as an imput. Older datasets (\eg\ Castle)
need then to be converted.

\begin{minted}{perl}
  #!/usr/bin/perl

  use strict;
  use warnings;

  while (<>) {
      chomp;
      my ($instr, $run_id, $lane, $tile, $x, $bases, $q_line) = split /\:/, $_;

      #Dots (uncalled bases) are translated to Ns
      $bases =~ tr/./N/;

      # Translate older Illumina's quality score to Phred 33
      $q_line =~ tr/\x40-\xff\x00-\x3f/\x21-\xe0\x21/;

      print "\@${instr}_$run_id:$lane:$tile:$x\n";
      print "$bases\n+\n$q_line\n";
  }
  end;
\end{minted}


\input{ds/vt.tex}
\input{ds/ibm.tex}
\input{ds/uhlen.tex}
\input{ds/gtex.tex}
\input{ds/pandey.tex}
\input{ds/kuster.tex}
\input{ds/cutler.tex}





\input{app/IntegSuppl.tex}
%\input{app/IntScatt.tex}

\chapter{List of publications}

\begin{refsection}[ownpubs]
    \small
    \nocite{*} % is local to the enclosing refsection
    \printbibliography[heading=none]
\end{refsection}

\endgroup
\bookmarksetup{startatroot}
\printbibliography[title={References}]

%\printglossaries\


\end{document}

