\chapter{Integrating gene expression data from undiseased tissues across RNA-Seq studies}%
\label{ch:Transcriptomics}

\setlength{\epigraphwidth}{0.54\textwidth}
\setlength{\epigraphrule}{0pt}
    \epigraphhead[5]{%
    \epigraph{\emph{So far, I think it's been working. But who knows?
    %I can see it now: me holding a map, scratching my head,
    %trying to figure out how I ended up on Venus.
    }}{Mark Watney~\mycite{AndyWeir}}
}


\vspace*{0.7in}

To pave the way towards
a generalised baseline expression reference for the \emph{normal}
(\ie\ non-disease) human,
I assess in this chapter the similarity
of the tissues sourced from different \Rnaseq\ studies and
the general profiles of their expressed genes.
When I started this project in 2013,
little was known of either the robustness or
the shortcomings and pitfalls of \Rnaseq\ and
its related processing of output data.
Since then, several studies were published assessing \Rnaseq\
robustness (See \Cref{sec:TranssCoop}).
A few are close in scope to my own investigations.
Thus whenever relevant,
I introduce and discuss my results in relation to the published ones.\mybr\

In the first part of this chapter,
I introduce the datasets based on the transcriptome studies
(described in \Cref{ch:datasets}) that I use in the different meta-analyses.
In the second part,
I appraise the congruence of the interstudy tissue expression profiles.
Then, I examine different components that may contribute
the most to (and thus explain)
the overall strong biological correlations that are observed
between the studies' tissues.
Finally, I explore the interstudy consistency of the gene expression profiles.\mybr\

All the work presented in this chapter was performed by myself under the
supervision of \alvis.
I received invaluable advice and help from my discussions with \nuno.
I also received general feedback and comments from \mar, \johan, \sarah, \gos\
and \wolfgang.\mybr\
\clearpage

\derivativeWork{}
\begin{itemize}[topsep=0pt,nosep]
    \item (paper)\fullcite{EBIgxa}
    \item (short talk) Quantitative Genomics 2015 --- Integration of
        independent human RNA-seq datasets: a feasibility study
    \item (poster) ECCB 2014 --- A feasibility study:
        Integration of independent RNAseq datasets
    \item (poster) SymBLS 2014 --- Integration of independent human
        RNAseq datasets, a feasibility study
    \item (invited short talk) GM$^2$ 2013 --- Baseline Gene expression Atlas
    \item (flash talk) CSAMA 2013 --- How quantitative is RNA-seq?
\end{itemize}
\clearpage

In the past years,
\Rnaseq\ rapidly gained popularity
for human gene expression studies
due to a broader dynamic range than previous technologies
and the promise to enable quantitative profiling \mycite{Marioni2008-xr}.
That technology was an advancement with respect to microarray assays
that are semiquantitative~\mycite{lee:2006}
and very prone to batch effects~\mycite{Irizarry2005-bs}.
However, \Rnaseq\ studies had shown variation in their conclusions on various
occasions for similar research topics~\mycite{seqcmaqc}.
At the time that I started my \phd,
it appeared that
\Rnaseq\ might share at least partially the problems encountered
with microarray assays.
In fact, \emph{batch effects} restrain the use of direct approaches
for the comparison of independent microarray data,
and the resulting insights are usually limited \mycite{Walsh2015-nf,%
Chrominski2015-ke,Rung2013-ul,Lazar2013-lj}.\mybr\
\vspace{-1mm}

The following meta-analyses attempt to provide more insights
into the interstudy \Rnaseq\ robustness for tissues expression
as a supporting exploratory study to the \egxa{}~\mycite{EBIgxa}.

\section{Meta-analyses' combined datasets}
\vspace{-2mm}
Through this chapter meta-analyses, I use two sets
based on combined subsets of the transcriptomic studies introduced
in \Cref{ch:datasets}.

The following \Cref{subsec:transtissueOverlap,subsec:transGeneOverlap}
illustrate the construction of these sets.\mybr\

While many approaches exist,
I usually consider the most stringent routes,
\ie\ I rather exclude part of the data to infer conclusions than
keep wider datasets and more partial, biased or ambiguous results.
Thus, I identified the identical core of
explored tissues and expressed genes across the studies.
From this base, I created two more robust combined datasets
(\setOne\ and \setTwo) for the meta-analyses.\mybr\

\subsection{Tissue overlaps across available normal human RNA-Seq studies \quad}%
\label{subsec:transtissueOverlap}
\vspace{-7mm}
\Cref{fig:VennStudiesT} presents the tissue overlap between the five studies.
All studies share at least four tissues:
\heart, \kidney, \liver\ and \testis.
This 4-tissue set is the base of the first combined dataset (\setOne).\mybr\

The greatest number of shared tissues occurs
between the two most recent studies,
\uhlen~\mycite{Uhlen2015} and \gtex~\mycite{GTExTranscript}.
This 23-tissue set is the base of the second combined dataset (\setTwo) and
includes
\Adipose, \Adrenal, \tissue{Bladder}\footnote{May also be referred to
as \Urinarybladder},
\Cortex, \hcolon, \Esophagus,
\Fallopian, \heart, \kidney, \liver, \lung, \Ovary, \Pancreas, \Prostate,
\salivary, \skeletal, \skin, \intestine, \spleen, \stomach, \testis,
\thyroid\ and \uterus.\mybr\

\begin{figure}[!htbp]
\includegraphics[scale=0.49]{transcriptomics/TransVennTissue.pdf}\centering
\caption[Distribution of unique and shared tissues between the
transcriptomic studies]
{\label{fig:VennStudiesT}\textbf{Distribution of unique and shared tissues
between the transcriptomic studies.} The five studies share four
common tissues: \heart, \kidney, \liver\ and \testis.
The most prominent overlap of tissues (23) is between \uhlen\ and \gtex.
}
\end{figure}
%\vspace*{-2.5mm}

\subsection{Common measured genes for each of the main shared-tissue sets\quad}%
\label{subsec:transGeneOverlap}
\vspace*{-9mm}
In the following sections of the thesis,
I only present the results based on the \htseq\ quantification.

As shown in \Crefp{tab:Trans5DF}{},
many of the transcriptomic studies I use were produced through
polyA-selected library protocols.
Hence,
to avoid unnecessary biases\footnote{See
\Cref{subsec:protcodingOnly}: \nameref{subsec:protcodingOnly}.},
I have limited my analyses to \pcgs\ (\ens{76}).
All mitochondrial genes have been filtered out before any \treps{} analysis
(as specified in \Cref{subsec:mito}).\mybr\

\begin{figure}[!hptb]
    \centering
    \begin{subfigure}[b]{\textwidth}
        \centering \includegraphics[scale=0.6]{transcriptomics/PcodingGenesExpressed1_4tissues.pdf}
        \caption{Four common tissues across the five tissues (\setOne)}\label{fig:ExpGenePcoding1}
    \end{subfigure}

    \begin{subfigure}[b]{\textwidth}
        \centering \includegraphics[scale=0.55]{transcriptomics/vennTissue23_1protcodgenes.pdf}
        \caption{Twenty-three common tissues\\ between Uhlén \etal\
        and GTEx studies (\setTwo)}\label{fig:ExpGenePcoding1_t23}
    \end{subfigure}
    \caption[Unique and shared \pcgs\ expressed (≥1 \FPKM) across RNA-Seq studies]%
    {\textbf{Unique and shared \pcgs\ expressed (≥ 1 \FPKM) across the \Rnaseq\ studies
    for \setOne\ and \setTwo}}
\end{figure}

The Venn diagram presented in \Cref{fig:ExpGenePcoding1} only includes \pcgs\
that are observed\footnote{See
\Cref{sec:ExpressedOrNot}: \nameref{sec:ExpressedOrNot}.}
at least once at 1 \FPKM\ for one of the four shared tissues.
As mentioned in the previous chapter (\Cref{subsec:averagedTissue}~p.~\pageref{def:trep}),
the bulk of expressed genes at this threshold is common
to all five studies.
While each study presents a tiny portion of genes
that are unique,
overall most genes are detected in at least two studies.
The most considerable contingent of shared gene expression is observed
between \uhlen\ and \gtex.\mybr\

\Cref{fig:ExpGenePcoding1_t23} presents a similar Venn diagram
which focuses on the set of twenty-three tissues (\setTwo)
between \uhlen\ and \gtex\ studies.
The uniquely expressed genes in each study are negligible compared to the overlap.
They represent at most 0.03\% of the measured genes in each of the studies.\mybr\

I have also analysed all the other subgroups of genes
(\ie\ unique to each study or shared only between two to four studies)
for any functional annotation enrichment (see \Cref{sec:enrichmentAnalysis}).
No analysis provided any conclusive result.\mybr\

\subsection{Combined datasets summary}

%\begin{itemize}[topsep=0pt,nosep]
%\item \setOne: 4 tissues --- 12,268 genes across the 5 \Rnaseq\ studies, and
%\item \setTwo: 23 tissues --- 17,551 genes across 2 of these studies.
%\end{itemize}

I have based all the meta-analyses of this chapter
on the two \setOne\ and \setTwo\ datasets,
which are defined as follow:
\begin{equation*}
        \setOneMath: \mathcal{D}_{\text{Trans}_1} \times
                     \mathcal{G}_{\text{protein coding}_1} \times
                     \mathcal{T}_1
\end{equation*}
and
\begin{equation*}
        \setTwoMath: \mathcal{D}_{\text{Trans}_2} \times
                     \mathcal{G}_{\text{protein coding}_2} \times
                     \mathcal{T}_2
\end{equation*}


where:
\quad\begin{eqlist}[\setlength{\itemsep}{0em}%
    \setlength{\topsep}{0em}%
    \setlength{\partopsep}{0em}%
    \setlength{\parskip}{0em}%
    \setlength{\parsep}{0em}]
    \item[\textbullet\ $\mathcal{D}_{\text{Trans}_i}$] is a set of
        \mRNA\ expression studies (presented in \Cref{sec:rnaseq-data}).\\
        With $\mathcal{D}_{\text{Trans}_1}$ = $\{$ \castle, \brawand, \ibm,
        \uhlen, \gtex$\}$ and \\
        $\mathcal{D}_{\text{Trans}_2} =$ $\{$ \uhlen, \gtex$\}$
    \item[\textbullet\ $\mathcal{G}_{\text{protein coding}_i}$] is a set of
        genes $g_{pc}$ that are shared by all elements of
        $\mathcal{D}_{\text{Trans}_i}$ and\\
        have a biotype described as \emph{protein coding} in \ens{76}.\\
        $\mathcal{G}_{\text{protein coding}_1}$ comprises 12,268 \pcgs.\\
        $\mathcal{G}_{\text{protein coding}_2}$ comprises 17,551 \pcgs.
    \item[\textbullet\ $\mathcal{T}_i$] is a set of
        tissues that are shared by all elements of
        $\mathcal{D}_{\text{Trans}_i}$.\\
        $\mathcal{T}_1$ includes four tissues and
        $\mathcal{T}_2$ twenty-three tissues.
\end{eqlist}

Note that as stated in \Cref{subsec:averagedTissue},
to avoid an unbalanced number of samples per tissues across studies,
I aggregate into a single virtual reference, \ie\ \trep,
the gene expression measured for each gene for each tissue in each study,
regardless of the number of replicates.
Thus, \setOne\ comprises 20 \treps,
and \setTwo\ 46 \treps.


\section{Prevalence of biological signal over technical variabilities at
tissue-level}\label{sec:Trans_ReproExpresTissue}

As shown in \Cref{ch:expression},
the expression levels of biological replicates (\ie\ identical tissue samples)
are highly correlated within the same study
and allow one to group the samples based on their biological source.
Thus, clustering the samples across studies should offer a rough assessment of
the underlying driving forces for the observed gene expression levels.
A clustering by study would mean that technical variabilities are stronger
than any biological expression signature
(which is an actual recurrent observation with microarray studies
due to their strong batch effects \mycite{Sudmant2015-zt}).
On the other hand,
an interstudy sample clustering by tissue would imply that \Rnaseq\ measurements
demonstrate a good (biological) signal over (technical) noise ratio.
In other words,
\Rnaseq\ would be then less prone to batch effects and more robust than
microarray assays~\mycite{Taminau2014-hr,Walsh2015-nf}.\mybr\

The heatmaps of the hierarchical clustering
of the \treps{}\footnote{\Glsxtrlong{TREP}. See \Cref{subsec:averagedTissue}:
\nameref{subsec:averagedTissue}.}
for \setOne\ and \setTwo\
are respectively presented in \Cref{fig:noMitoNoRep4T,fig:noMitoNoRep23T}.
They are based on clustering (Ward's method linkage \mycite{Ward1963})
the \treps{}' Pearson correlation coefficients
(\pcgs\ expressed at least at 1 \FPKM).\mybr\

Both heatmaps show that
the overall biological signal measured in the tissues is stronger than
the noise generated by any technical variation or batch effect.

In \Cref{fig:noMitoNoRep4T},
each cluster corresponds to a tissue.
The clustering signal by tissue dominates over the signal from the dataset.
It highlights a greater biological similarity of the \treps\
due to their sampling origins rather than any possible
technical similarity due to laboratory protocol variations.\mybr\

One may object that
the very different gene expression landscapes of \Heart, \Kidney, \Liver\ and \Testis\
\mycite{ramskoldan:2009,Lukk2010-op,Danielsson2015-cn,Sudmant2015-zt,GTExTranscript,Uhlen2015}
may drive this result
and lesser differentiated tissues may exhibit more mitigated correlations.
\Cref{fig:noMitoNoRep23T} (\setTwo) confirms that the biological origin of the tissues
is the dominant criterion for the clustering of the \treps.\mybr\

Moreover, in many cases, \treps\ mixtures occur in close biologically related tissues,
\eg\ \fallopian\ and \Ovary\ or
\salivary\ with \Esophagus\ and \Stomach\ \treps.
The functional proximity of these tissues is likely supported
by an overall similarity in their gene expression.
Thus, even though there are clear biological substructures emerging
like for \Heart\ and \Skeletal,
without correction, the biological signal to technical noise ratio
for close tissues may be insufficient to discriminate them accurately in every case.

The general observed biological prevalence holds
when I extend the analysis to include all the available
tissue samples (see \Cref{fig:noMitoRep4T} and \Cref{fig:noMitoRep23T}).
See also \Crefu{sec:rnaseq-data} and \Crefu{tab:repCorr}.\mybr\

\begin{figure}[!htpb]
    \includegraphics[scale=0.84]{transcriptomics/heatmap4TnoMitonoRep_1TC.pdf}\centering
    \caption[Heatmap of the 4 common tissues across the 5 studies]%
    {\label{fig:noMitoNoRep4T}\textbf{Heatmap of the four common tissues
    across the five studies.}\\All \pcgs\ (except the mitochondrial
    ones) at least expressed at 1 \FPKM\ are included.\\All the
    different \treps\ cluster by tissue of origin
    regardless of their study sources.
    Each of the colours on the top bar following the x-axis
    is associated to one of the study
    (purple for \uhlen, blue for \brawand, green for \gtex,
    orange for \castle{} and red for \ibm{}),
    and the colours on the side bar following the y-axis
    are associated to the tissues
    (green for \kidney, red for \heart, blue for \testis\
    and orange \liver).}
\end{figure}

\begin{figure}[!htpb]
    \cpv{\includegraphics[scale=0.84]{transcriptomics/heatmap23TnoMitonoRep_1TCcpv.pdf}}\centering
    \caption[Heatmap of 23 common tissues between Uhlén and GTEx studies]%
    {\label{fig:noMitoNoRep23T}%
    \textbf{Heatmap of twenty-three common tissues between Uhlén and GTEx studies.}
    All \pcgs\ (≥ 1 \FPKM\ with the exclusion of the mitochondrial
    genes) are included.\\Most \treps\ cluster by tissues (y-axis colour bar)
    rather than by study of origin (x-axis colour bar)
    with a few exceptions:
    there is a mixture of the \fallopian\
    and \Ovary\ \treps.
    In addition, \Salivary\ \treps\ is more correlated to
    \Oesophagus\ or \Stomach\ regarding the original study.
    \Bladder\ \treps\ seem to cluster randomly with the others.
    However, these \treps\ are in singleton groups.}
\end{figure}

\Cref{fig:SamedistribPearsCorr} shows the distribution of the Pearson correlation
coefficients for the pairs of the profiles (\treps) of tissues with the same name
sourced from the different studies
for both of the combined datasets \setOne\ (4-tissue set)
and \setTwo\ (23-tissue set).
Even with the lack of any batch effect correction,
most of the Pearson correlations are above 0.5.
There are two exceptions: the
correlation between the \Testis\ \treps\ of \castle\ and \vt\ ($r$ = 0.42)
from \setOne\ and
\Salivary\ \treps\ of \uhlen\ and \gtex\ ($r$ = 0.2)
from \setTwo.
The median correlation for \setOne\ is
about 0.7 and 0.84 for \setTwo.
Spearman correlation gives even better results:
average correlation coefficients are $\rho$ = 0.49 for \setOne\
and $\rho_{\text{\setTwo}}$ = 0.9;
the median correlations are $\rho_{\text{\setOne}}$ = 0.88
and $\rho_{\text{\setTwo}}$ = 0.93.

Both\label{seg:betterTreps} the
Pearson\footnote{Despite one major outlier in the second
combined dataset (\tissue{Salivary gland} --- Pearson correlation: $r$=0.2)} and the
Spearman correlation coefficients for the more exhaustive \setTwo\ set,
which comprises the two most recent studies,
are higher than the observed correlations for \setOne.
Three main reasons may explain this situation as they contribute to lower
the technical variations:\mybr\
\vspace{-3mm}
\begin{itemize}[topsep=0.5pt,nosep]
    \item In addition to using paired-end sequencing,
        the library preparation protocols were better established
        for these two more recent studies;
    \item The instrument used for the sequencing were
        from the same series (HiSeq~2000 and HiSeq~2500); and
    \item These studies present a higher number of replicates per tissue.
\end{itemize}

\begin{figure}[!htpb]
    \includegraphics[scale=0.55]%
{transcriptomics/TransPearsonDistributionIdenticalOnly.pdf}\centering
\caption[Distribution of the correlations of same tissue pairs for the 4 and 23
tissues combined datasets.]{\label{fig:SamedistribPearsCorr}\textbf{Distribution
of the Pearson correlation coefficients of same tissues pairs for the four and
the twenty-three tissues combined datasets.}
In general, the Pearson correlations are high when we are
\emph{directly} comparing \treps\ from different studies.\\
The same-tissue pairs in the 23-tissues combined dataset (\setTwo) present
a higher median correlation ($0.85$)
and narrower distribution than
in the 4-tissues combined dataset (\setOne) (median$ = 0.74$).
However, \setTwo\ displays one outlier with
a very low Pearson correlation ($0.2$: \salivary\ tissue).
Sampling, processing differences or biological reasons
may just as well explain this outlier.}
\end{figure}

On the other hand,
the pairs comprising different tissues are very lowly correlated in general
(see \Cref{fig:distribCorr}).
Although, in a few cases of \setTwo\ (23-tissues combined dataset),
high correlations are also observed for different-tissues pairs,
\eg\ \Fallopian\ and \Uterus\ from \gtex\ study
(see also \Cref{fig:noMitoNoRep23T}).
It is rather hard to decipher if this may be due to a technical issue
(\eg\ at the collection or library preparation stage)
or because these tissues are biologically very close.\mybr\

As the exclusion of the \emph{undefined}\footnote{I.e.\
\emph{unobserved} --- See also
\Cref{subsec:ExpressedOrNot-undefined}: \nameref{subsec:ExpressedOrNot-undefined}}
genes from the analyses handles
one possible source of spurious Pearson correlation (due to null values),
I have then checked, and, discarded,
another possible source that is the skewed distributions;
highest expressed genes may be technical artefacts,
but they fail to show any significant correlation
(see \Cref{subsec:mrnaHighExp-app}).\mybr\

The high correlations are the results of the overall similarity
of genes expression patterns across tissues and studies.\mybr\

\section{Global~stability~of~gene~expression~profiles~across~studies}

After validating that \Rnaseq\ allows distinguishing the shared biological origin
of most tissues (\treps) across different studies,
the question then arises as to how consistent is
the expression of each gene for a given tissue between studies.\mybr\

To this aim,
I first assess the expression variability of the genes
across the studies.
Then, I explore the interstudy coherence of several gene categories.
I focus on the tissue-specific (TS) genes,
and on a larger number of categories developed upon
classifications from \uhlen{}'s laboratory.\mybr\


\subsection{Genes with tissue-specific (TS) expression}\label{sub:TisSpeGene}
%\sectionmark{Tissue specific genes}

Tissue-specific (TS) genes are arguably the genes
that ought to present a robust expression profile across studies.

\minisec{Tissue specificity definition}
The definition of tissue specificity varies from one study to another.
See also \citet{Santos2015-rj}.
\citet{Liang2006-mk} that define \emph{tissue specificity}
only for genes expressed solely in one tissue,
and then \emph{tissue selectivity} for genes expressed in more than one tissue
with an expression enriched in one or a subset of tissues.
Other studies have a broader definition of tissue specificity.
They identify genes above a given threshold of tissue selectivity (or enrichment)
as tissue-specific genes
(\eg\ \citet{Uhlen2014,Jiang2016-sv}).
In this second case,
genes with a single-tissue expression are an extreme case of \gls{TS} genes.\mybr\

Within this thesis, I use the second definition,
\ie\ I consider genes as \gls{TS}
as long as they display a higher tissue selectivity than a preset threshold,
regardless of how many tissues express them.
Indeed, every study presents a subset of genes
that are expressed above 1 \FPKM\
in a sole tissue (see \Cref{fig:breadthGenesP1}).
However, the decreasing number of these genes
when increasing the number of considered tissues highlights
the arbitrary relativity introduced by the study design.\mybr\

For this definition,
there are many methods to characterise genes tissue-specificity
(\eg\ \citet{Cavalli2011-bo,Xiao2010-mz,Karthik2016-mu,Kim2017-dz,%
Kryuchkova-Mostacci2017-mk,Kadota2006-eb,Yu2006-ha,Martinez2008-bm}).
There are also databases that record previously identified \gls{TS} genes,
for normal conditions,
\eg\ \WebFoCi{TiGER}{http://bioinfo.wilmer.jhu.edu/tiger/}{tiger} or
\WebFoCi{TiSGeD}{http://bioinf.xmu.edu.cn:8080/databases/TiSGeD/index.html}{Xiao2010-mz}
and more specialised ones, \eg\ for cancer
\WebFoCi{TissGDB}{https://bioinfo.uth.edu/TissGDB/index.html}{Kim2017-dz}.\mybr\

\minisec{TS genes characterisation approaches used in this thesis}
From the possible approaches to characterise the \gls{TS} \pcgs,
I detail three that I used in the following subsections.
First, I have queried \gls{TIGER} to capitalise on previous knowledge.
Then, to derive the \gls{TS} genes directly from \setOne\ and \setTwo,
I have used a published method,
that uses the gene expression \emph{fold change} (FC) ratio across the tissues.
Finally, I have employed a robust method designed to detect outliers in data,
\ie\ Hampel's test \mycite{Hampel1974},
to identify genes which present an unusual expression level in a single tissue.
In fact, as gene tissue selectivity and tissue specificity definitions are
relative to a context,
if the latter changes,
the genes attributes may change as well
(\eg\ one gene that is \cpv{non-specific} in \setTwo\ may be \heart{}-specific in \setOne).\mybr\


\subsubsection{Use of prior knowledge: TiGER database}\label{subsub:Tiger}

\gls{TIGER}~\mycite{tiger} reports \gls{TS} genes for thirty independent tissues
(based on \glspl{EST} experiments).\mybr\

After retrieving the list of genes for all reported tissues,
I have mapped the \gls{Refseq} identifiers provided by \gls{TIGER}
to \gls{Ensembl} gene identifiers (\hg{38}, \ens{76}).
Then, I removed all duplicates due to the identifier translation within each tissue,
and I also filtered out all the genes identifiers that
I found in more than one tissue:
\gls{TIGER} lists a subset of the same genes in many tissues,
but the modification in the annotation may also explain part of the
repetitive genes.
Thus, for each tissue,
I have a list of identifiers that are specific to that tissue only.\mybr\

\begin{figure}[!ht]
    \includegraphics[scale=0.9]{transcriptomics/Tiger5DF4Tissues.pdf}\centering
    \vspace{-.15in}
    \caption[Expression heatmap of the four tissues across the five datasets based on
    TiGER]{\label{fig:TigerGenes}\textbf{Expression heatmap of the four tissues
    across the five datasets based on \gls{TIGER} information.}
    This heatmap illustrates three subsets of genes:
    genes for which real expression data confirm their \gls{TIGER} definition;
    genes failing to show any \gls{TS} profile in their expression data; and
    genes with mismatching tissue specificity between \gls{TIGER}
    definition and the expression data.
    The colourbar above the heatmap is representing the tissue
    for which \gls{TIGER} annotates the genes (presented as columns) as \gls{TS}
    (red for \Heart, green for \kidney, light orange for \liver\ and blue for \testis).
    \gls{TIGER} definitions are of variable accuracy.
    }
\end{figure}

\Cref{fig:TigerGenes} is an expression heatmap based on
a subset (\ie\ $916$)  of \pcgs\
that are present in this final list of translated \gls{TIGER} genes
for the  \heart, \kidney, \liver\ and \testis.
There are three main types of genes.
The largest group comprises the genes with a corroborating profile
between the \gls{TIGER} definition and the real data.
Then, a second smaller group encompasses genes
listed as \gls{TS} in \gls{TIGER},
but fails to demonstrate expression specificity
towards any tissue in the real data.
Finally, the third group includes a very tiny subset of genes which
are more specific to another tissue than the initially stated one.\mybr\

Thus, without any additional knowledge,
it is difficult to predict beforehand which original \gls{TIGER} definitions
will be confirmed or rejected by expression data.
Remarkably, most of the genes present the same expression pattern
through the tissues across each of the studies
and thus regardless of their \gls{TIGER} category.
Once again, \castle\ expression data is exhibiting
the only few observed discrepancies\footnote{Reminder:
the \gls{FPKM} quantification (used here) is sensitive
to the number of identified genes (see \Cref{eq:rpkm-fx}~\vref{eq:rpkm-fx})
and \castle\ study identifies and quantifies many more \glspl{RNA} than the
other studies as it uses a whole \RNA\ protocol
while the others are using polyA-enrichment (see \Cref{ch:datasets}).}.\mybr\

\begin{figure}[!htb]
    \includegraphics[scale=0.9]{transcriptomics/ConceptOverlap.png}\centering
    \caption[Overview for the comparison of the genes across the five
    studies based on a ranked descriptor 5 studies]{\label{fig:overlapConcept}%
    \textbf{Overview for the comparison of the genes across the five
    studies based on a ranked descriptor.}
    The first step applies individually to each of the studies
    within the combined dataset (\ie\ here \setOne).
    It consists in extracting a single value per gene
    (\eg\ a statistic or any other quantitative descriptor)
    either for the entire \emph{d}ataset (referred thereafter as \emph{D-approach}) or
    for each \emph{t}issue in each dataset (referred as \emph{T-approach}).
    The next steps include
    computing (cumulatively) the intersection size number for each rank
    and plotting this number divided by the rank
    as a function of the number of considered genes (\ie\ rank).}
\end{figure}

\subsubsection{Fold change method}\label{subsub:TisSpeGeneMethodPerso}
As \citet{DESeq2} noted the most common approach for detecting
a gene expression difference between two conditions is
to study the expression fold change (FC) ratio between these conditions.
This method is still broadly present in the literature,
especially for studies other than differential gene expression analyses\footnote{%
Studies comparing gene expression of a treated or diseased condition
to control samples};
as examples, see~\citet{Uhlen2015,Zhu2016-xo,Yu2015-uh}.
Besides, \egxa\ \mycite{EBIgxa} relies on this method to select
the most specific genes for \emph{baseline} studies\footnote{In
contrast to differential gene studies,
the baseline studies focus on
depicting the expression landscape of each covered condition
instead of focusing on the gene expression through these conditions.}
(see \Cref{fig:gxaEx}).
There are also a few variations
on how to compute this ratio; see~\citet{Zhu2016-xo,Uhlen2015}.\mybr\

In this thesis,
I compute the FC ratio by dividing the expression
of each gene in a given tissue
by the average expression of this gene across all the other tissues of that study
in the combined dataset.\mybr\

\begin{minipage}{\textwidth}
\begin{equation}
    \tag{Fold change (FC) ratio}
    %\FCm_{g,t,d}=\frac{x_{g,t,d}}{\max\limits_{t' \in \mathcal{T}} x_{g,t',d}}
    \FCm_{g,t,d}=\frac{x_{g,t,d}}{\frac{1}{n}\sum\limits_{i=1}^n {x_{g,i,d}}}
\end{equation}
where:\quad\begin{eqlist}[\setlength{\itemsep}{0em}%
            \setlength{\topsep}{0em}%
            \setlength{\partopsep}{0em}%
            \setlength{\parskip}{0em}%
            \setlength{\parsep}{0em}]
        %\item[\textbullet\ $\mathcal{T}$] is a set of tissues $t$
        \item[\textbullet\ $x$] is the expression value of the gene $g$
            in the tissue $t$ in a study $d$
        \item[\textbullet\ $n$] is the number of tissues $t$
\end{eqlist}
\end{minipage}

Studies usually pick arbitrary cut-offs to characterise the specific genes.
\citet{Uhlen2015} uses two-fold and five-fold cut-offs
to determine \emph{enriched} and \emph{enhanced} tissue genes.
\citet{Zhu2016-xo} set their cut-off at $2$ to characterise \gls{TS} protein-coding
and noncoding transcripts.
However,
I avoid arbitrary cut-offs, and
I use the FC ratio to rank the \pcgs\ of my combined datasets according
to their specificity within each tissue:
higher FC ratios indicate genes with higher specificity.
I then assess the consistency of the tissue specificity of the genes through the
various studies.
For that, I have followed the \emph{T-approach} overviewed in \Cref{fig:overlapConcept}.\mybr\

\begin{figure}[!thb]
    \includegraphics[scale=0.8]{transcriptomics/mostSpe4TP1.png}\centering
    \vspace{-0.2in}
    \caption[Intersection size curve of \setOne\ genes based on their FC ratio
    rank in each tissue across the five studies]{\label{fig:mostSpe4T}\textbf{Intersection
    size curve of \setOne\ genes based on their specificity (FC ratio rank)
    in each tissue across the five studies.}
    When ranked by specificity in \heart, \kidney\ and \liver,
    one fortieth of \setOne\ \pcgs\ are commonly shared between the five studies.
    For \testis, the shared amount of genes reaches more than one-tenth of \setOne\
    whole set of genes.
    Compared to the most variable genes (\Cref{fig:cvEvol5DF}),
    the most specific genes seem to be more consistent across the studies.
    }
\end{figure}

\Cref{fig:mostSpe4T} presents the shifts in the intersection size curves of the
four tissues of \setOne.
The most specific genes in each tissue of \setOne\ are shared between the
five studies.
Indeed, the slopes are very sharp before reaching a peak and dropping as sharply
for the first fortieth genes in \heart, \kidney\ and \liver.
The intersection of the most specific genes is even greater for \testis\ as
it concerns more than a tenth of \setOne\ genes.
\Cref{fig:mostSpe23T} shows that this observation holds true for \setTwo\
when the number of tissues and genes is increased.\mybr\

\begin{figure}[!htbp]
    \includegraphics[scale=0.7]{transcriptomics/mostSpe23TP1.png}\centering
    \vspace{-0.2in}
    \caption[Intersection size curve of \setTwo\ genes based on their FC ratio
    rank in each tissue across the two studies]{\label{fig:mostSpe23T}\textbf{Intersection
    size curve of \setTwo\ genes based on their specificity (FC ratio rank)
    in each of the twenty-three tissues across the two studies.}
    }
\end{figure}

\subsubsection{Hampel's test: detection of \emph{atypical} expression}\label{subsub:Hampel}

The last method I used to characterise the \gls{TS} genes is the Hampel's test.
This test is a robust method
for detecting outliers~\mycite{Davies1993-nv,Pearson2002-im}
in data that are identically and independently distributed (i.i.d.)~\mycite{Liu2004-kf},
while easy to implement and use~\mycite{LinsingerHampel}.
Much interlaboratory or interstudy research in the literature
\eg~\citet{LinsingerHampel,Lewczuk2006-wq,Rocke1983-qa,Apfalter1999-ca}
use the Hampel's test to detect outliers.
The method uses the median and the \gls{MAD}\footnote{MAD:~median absolute deviation}
to estimate the location and the spread,
and a cut-off to define the observations that stand apart.\mybr\

\begin{figure}[!htb]
    \includegraphics[scale=1]{transcriptomics/hampel5DF4Tissues.pdf}\centering
    \caption[Expression of the genes picked with Hampel method]{\label{fig:hampelExp}%
    \textbf{Expression of the genes picked consistently with the Hampel method
    in each study solely in one tissue.}}
\end{figure}

For this thesis, I have derived this test to identify conditions (\ie\ tissues)
where the gene expression is \emph{atypical}.
I rely on the two facts that
most genes are expressed everywhere \mycite{ramskoldan:2009,Uhlen2015,GTExTranscript}
(see also \Cref{fig:breadthGenesP1}),
and they mostly present a limited variation in their expression
through the various tissues (see \Cref{fig:HistCV4T,fig:HistCV23T}).
There is a tissue specificity for a gene when its expression to this tissue is atypical,
\ie\ the expression in this tissue is an outlier to the average expression profile
across the other tissues.
Besides,
this test allows detecting genes that are over- or under-expressed in specific tissues,
whereas the other methods are only detecting the overexpressed genes.\mybr\

After implementing the method (see \Cref{algo:hampel}, p.~\pageref{algo:hampel})
with a (widespread) adimensional cut-off of $5.2$, % (which assumes a normal-like distribution),
I have applied it to \setTwo\ and the whole original datasets.
\setOne\ comprises too few tissues to allow detecting \emph{atypical} expression.
Overall, there are always more than 60\% of congruence between the genes tagged
as atypical in a specific tissue for \setTwo\ and the whole dataset.
The proportion of agreement between the partial and whole datasets increases
when I filter the results to keep only the genes that are recurrently
picked for both \uhlen\ \etal\ and \gtex\ data.\mybr\

\Cref{fig:hampelExp} regroups the genes that the Hampel test detects
as outliers for the five studies for their four shared tissues.
All corresponding-tissue \treps\ present similar patterns of expression
regardless of their original study,
although the \castle\ \treps\ have overall lower expression values than the others.
Other filters may improve the results.\mybr\

Overall, the \gls{TS} genes,
identified separately in each dataset and with several methods,
are showing a cleaner biological interstudy signal
over possible technical intrastudy noise
and are contributing to the high interstudy tissue correlations
presented above.\mybr\

\subsection{Uhlén categories}\label{sub:UhlenGeneCat}

Uhlén laboratory publications \mycite{Uhlen2014,Uhlen2015,Uhlen:2016}
use different categories of genes to describe the normal human transcriptome.
As their classification changes between these related papers,
I have redefined a classification based on them
(presented in \Cref{tab:UhlenCat})
before applying it to \setOne\ and \setTwo\ (\Cref{tab:UhlenCategoriesProtCoding}).\mybr\

The following classification considers the breadth\footnote{The breadth
of expression of a gene is the number of tissues (or cell lines)
in which it is expressed.
},
the level and the specificity of the gene expression.

\begin{table}[!htpb]
\centering
\caption[Gene classification]{\textbf{Gene classification}\label{tab:UhlenCat}\\
\footnotesize{adaptation of \uhlen\ \etal\ classification~\mycite{Uhlen2014,Uhlen2015,Uhlen:2016}}}
\begin{tabular}{@{}ll@{}}
\toprule
Category        & Definition                                                                                           \\ \midrule
Not detected    & Never detected above 0 FPKM                                                                    \\
Not expressed   & Never detected above 1 FPKM                                                                    \\
Mixed high      & Expressed in a subset of tissues and always $\geq 10$ FPKM                                     \\
Mixed Low       & Expressed in a subset of tissues and always $< 10$ FPKM                                 \\
Ubiquitous High & Expressed in all the tissues and always $\geq$ 10 FPKM                                         \\
Ubiquitous Low  & Expressed in all the tissues and always $< 10$ FPKM                                     \\
Group enhanced  & Expressed in a subset of tissues with an expression $\geq 5*mean_{all~the~tissues}$             \\
Tissue enhanced & Expressed in a single tissue with an expression $\geq 5*mean_{all~the~tissues}$                 \\
Tissue enriched & Expressed in a single tissue with an expression $\geq 5*Max_{all~the~other~tissues}$ \\ \bottomrule
\end{tabular}
\end{table}

\Cref{tab:UhlenCategoriesProtCoding} shows that for many of these categories,
the number of shared \pcgs\ is high between the different studies of the
two combined datasets \setOne\ and \setTwo.
It supports the previous results that \pcgs\ present in general a similar
gene expression profile for a same set of tissues across studies.\mybr\

\begin{sidewaystable}
\centering
\caption[Uhlén \etal\ gene categories]{\label{tab:UhlenCategoriesProtCoding}%
\textbf{Uhlén \etal\ gene categories}\\
\footnotesize{Apart from the undetected genes and the ones expressed below 1 \FPKM,
a gene may be referenced in several categories.}}

%\begin{tabular}{@{}lllllllllll@{}}
\begin{tabular}{@{}ccccccccccc@{}}
\toprule
\multicolumn{2}{c}{\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\ens{76}
    \\($\sim$22,500 protein\\coding genes) \end{tabular}}} &
\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Not\\detected\end{tabular}} &
\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}Not expressed\\ at 1 \gls{FPKM}\\
    cut-off\end{tabular}} &
\multicolumn{2}{c}{Mixed expression} &
\multicolumn{2}{c}{Ubiquitous expression} &
\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Group \\Enhanced\end{tabular}} &
    \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Tissue\\ Enhanced\end{tabular}} &
        \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Tissue\\ Enriched\end{tabular}} \\
    \cmidrule(lr){5-8}
\multicolumn{2}{c}{}
    &  &  &
    \begin{tabular}[c]{@{}c@{}}Low\\ (\textless\ 10 \gls{FPKM})\end{tabular} &
        \begin{tabular}[c]{@{}c@{}}High\\ (≥ 10 \gls{FPKM})\end{tabular} &
            \begin{tabular}[c]{@{}c@{}}Low\\ (\textless\ 10 \gls{FPKM})\end{tabular} &
    \begin{tabular}[c]{@{}c@{}}High\\ (≥ 10 FPKM)\end{tabular} &  &  &  \\
        \midrule
        \multicolumn{1}{c}{%
        \multirow{6}{*}{\rotatebox[origin=c]{90}{\parbox[c]{3cm}{\centering Whole
        dataset}}}} &
        Castle & 3,403 & 3,268 & 8,773 & 1,033  &
        1,399  & 634   & 11   & 3,664   & 1,975 \\
        \multicolumn{1}{c}{} & Brawand & 2,964 & 3,095 &
        8,034 & 1,788  & 1,760 & 958   & 0  &
        2,729  & 2,548 \\
        \multicolumn{1}{c}{} & IBM & 2,693 & 2,605  &
        7,325  & 1,406  & 1,135 & 858  & 322 &
        5,248  & 2,453  \\
        \multicolumn{1}{c}{} & Uhlén & 2,662 & 1,747 &
        5,769 & 1,053  & 456 & 406  & 2,511  &
        5,201  & 2,333  \\
        \multicolumn{1}{c}{} & GTEx & 2,197  & 1,886  &
        5,556  & 1,117 & 687  & 698  & 3,859  &
        4,356  & 1,919 \\ \cmidrule(l){2-11}
        \multicolumn{1}{c}{} & Consensus & 2,197 & 486  &
        1,749  & 221  & 33  & 161  & 0  & 677  &
        \begin{tabular}[c]{@{}c@{}}531 %$[$518$]$
        \end{tabular} \\
            %\multicolumn{1}{c}{} & \footnotesize{without Gtex} &
            %\footnotesize{2,413} & \footnotesize{638} & \footnotesize{2,152} &
            %\footnotesize{286} & \footnotesize{63}  & \footnotesize{179} &
            %\footnotesize{0}  & \footnotesize{814} &  \footnotesize{587}  \\
            \midrule
\multirow{6}{*}{\rotatebox[origin=c]{90}{\parbox[c]{3.5cm}{\centering Common\\
4-tissues\\combined\\datasets}}} &
Castle & 19,066 & 2,994 & 8,589 & 1,513 &
2,994 & 1094 & --- & --- & 2,185 \\
& Brawand & 19,505  & 2,962  & 8,626  & 2,228
& 2,962  & 1251 & --- & --- & 3,672  \\
& IBM & 19,776  & 2,989 & 8,534 & 1,954 &
2,989  & 1212  & --- & --- & 2,824  \\
& Uhlén & 19,807 & 2,917 & 8,367 & 2,227 &
2,917  & 1190  & --- & --- & 3,730  \\
& GTEx & 20,272 & 3,870 & 8,988  & 2,312 &
3,870  & 1427  & --- & --- & 3,554  \\
\cmidrule(l){2-11}
& Consensus & 1,973 & 550 & 3,351 & 649 &
550  & 439 & --- & --- & 1,412  \\
%& \footnotesize{without Gtex} & \footnotesize{2,413}  & \footnotesize{2,186}  &
%\footnotesize{3538}  & \footnotesize{639}  & \footnotesize{576}  &
%\footnotesize{439}  & \footnotesize{---} & \footnotesize{---} & \footnotesize{1,462}
\midrule
\multirow{3}{*}{\rotatebox[origin=c]{90}{\parbox[c]{1.7cm}{\centering
Common\\23-tissues\\combined datasets}}} & Uhlén & 2,662  & 1,970  &
6,160 & 1,135 & 594  & 427 & 1,285 &
5,776 & 2,518 \\
& GTEx & 2,197 & 2,258 & 6,966  & 1,540 &
1,822  & 997 & 1,048 & 5,496  & 2,460 \\
\cmidrule(l){2-11}
& Consensus & 2,197 & 1,544 & 4,936 & 791 &
423 & 417 & 558 & 4,223 & 1,885 \\
\bottomrule
\end{tabular}
\end{sidewaystable}

\subsection{Similar expression variability of the genes across studies\quad}
\vspace{-5mm}
To further appraise the robustness of gene expression,
I have studied more globally their variability across studies.

There are several available estimators to describe the gene expression variability,
\eg\ the standard deviation~(sd) the variance~($sd^2$) or the coefficient of
variation~($\frac{sd}{mean}$).
I only report here the results based on the coefficients of variation.
The \gls{cv} allows assessing
the dispersion of the gene expression values
across the tissues within each dataset.
As it adjusts for the mean,
it is a more straightforward estimator to interpret than
the variance itself,
in particular for interstudy comparisons.\mybr\

As depicted in \Cref{fig:HistCV4T} (and \Cref{fig:HistCV23T} for \setTwo),
the distribution of gene expression \cvs\ presents a similar pattern
across the five studies of \setOne.\mybr\

The five datasets present two peaks.
One at approximately 0.5 which characterises genes
that are quite invariant in their expression
across the four tissues within each study.
Another subset of genes forms a peak for \cvs\ equal to 2.
This last group of genes are the most variable ones in each dataset.
There is an overlap of the most variable genes between the five datasets
(as shown in \Cref{fig:cvEvol5DF}).\mybr\

While \Cref{fig:noMitoNoRep23T} has already established that
expression profiles for each tissue are very similar across the studies,
\Cref{fig:HistCV23T} highlights that most genes seem to share the same
intertissue expression profile variability as the distribution of the coefficients
of variation across \uhlen\ \etal\ and \gtex\ studies are very alike.\mybr\

\begin{figure}[!htpb]
    \captionsetup{singlelinecheck=off}
    \includegraphics[scale=0.73]{transcriptomics/distributionCV_4commonTissues.pdf}%
    \centering
    \vspace{-4mm}
    \caption[Coefficients of variation across the 5 studies for the set of common
expressed genes and tissues]{\label{fig:HistCV4T}\textbf{Distribution of the
\cvs\ (cv) across \setOne\ (common set of expressed \pcgs\
across the four common tissues):
\{\Heart, \Kidney, \Liver, \Testis\}
across the five studies.}\\
The coefficients of variation (\gls{cv}) of the \pcgs\ (12,268) of the four tissues
present the same bimodal distribution profile across the five studies.
\\These profiles present two peaks: at $0.5$ and $2$.\\
Genes with a \gls{cv} less than or equal to $0.5$ have
a similar expression profile to a left-truncated version of
the complete gene set ones (due to the $1$ \FPKM\ cut-off)
as in \Cref{fig:distribPlot}.
On the other hand, the \pcgs\ with a coefficient of variation
equal to or greater than $1.5$ have two kinds of distinct profiles:
{\small
\begin{itemize}[topsep=0pt,nosep,leftmargin=95pt,listparindent=5pt]
    \item The gene expression is low across the four tissues, and
        it is above the cut-off of $1$ \FPKM\ only once
        (see \Cref{fig:mostVarBreadth}); or
    \item The gene expression is specifically high for one single tissue
        relative to the three others (See \Cref{fig:highestCVhist}).
\end{itemize}
}}
\end{figure}

\begin{figure}[!htbp]
    \includegraphics[scale=0.73]{transcriptomics/distributionCV_23commonTissues.pdf}\centering
    \vspace{-4mm}
    \caption[Coefficients of variation across \setTwo\ 2 studies and their set
    of common genes across the 23 tissues]{\label{fig:HistCV23T}\textbf{Distribution
    of the \cvs\ across \setTwo.}
    The bimodal distribution is more unbalanced than in \Cref{fig:HistCV4T}.
    Indeed, as more tissues are included for the calculation of the \cvs,
    the second peak is found around $5$.
    This peak has a smaller amplitude than the peaks at $2$ in \Cref{fig:HistCV23T}.
    There are still many genes that have a \cv\ around $2$.
    However, the overall distribution of the higher \cvs\ is smoother than
    for \setOne.
    Hence, most genes present a similar profile of expression through the various
    tissues.}
\end{figure}

More in-depth analyses confirmed that
overall the genes present an equivalent \cv\
from one study to another
for the same tissue set. See \Cref{fig:cvEvol5DF,fig:cvEvol2DF}.\mybr\

\subsection{Curated sets}\label{subsec:Trans_curatedSets}

Together the results from the previous sections indicate that
many genes categories (if not all of them) have
an equivalent (\ie\ stable) expression profile
across studies for the same tissues.

\Pcgs\ that were characterised consistently
as any of the aforementioned categories across the five datasets of \setOne\
or the two of \setTwo\
are provided digitally as supplementary data.
They can be found at \url{http://www.barzine.net/~mitra/thesis}.
\\The code required to produce these results can be found at \github.\mybr\

\section{Discussion}\label{sec:Trans_discussion}
%\vspace{-6mm}
In this chapter,
I have directly compared and integrated
the human tissue transcriptome from five \Rnaseq\ studies.
The meta-analyses are based on the largest number
of undiseased human tissue studies to date.
I have constructed two combined datasets of \pcgs.
The first one (\setOne) comprises four tissues
and 12,268 shared genes
extracted from five independent studies
(\cite{Krupp2012,VTpaper,Uhlen2015,GTExTranscript} and \ibm)
and the second one (\setTwo) comprises twenty-three tissues
and 17,554 shared genes
from two studies (\cite{Uhlen2015,GTExTranscript}).\mybr\

Clustering analyses (and \Welchttest) of these two sets
confirm that \Rnaseq\ technical noise is
lower than relevant biological signals present in the data.
Indeed,~\cite{Sudmant2015-zt,Danielsson2015-cn,Yu2015-uh} and~\cite{Uhlen:2016}
also observe that interstudy corresponding tissues pairs are more related than
intrastudy non-corresponding tissue ones
(average correlation for corresponding tissue-pairs $r=0.75$, $\rho=0.88$;
average for non-corresponding tissue-pairs $r=0.20$, $\rho=0.75$).\mybr\

I have then shown that overall genes present
similar interstudy expression variability profiles for the same tissue set.
I have considered different gene groups
to examine their coherence of expression profiles more closely.

Since there is no generally accepted definition
of a \gls{TS} gene (see \Cref{sub:TisSpeGene}),
I have relied on three different methods to study them,
including extracting \gls{TS} gene definitions
from an existent resource
\WebCi{TiGER}{http://bioinfo.wilmer.jhu.edu/tiger/}{tiger}
that I have updated to the current human genome build (\hg{38}).
Mining the experimental data with this updated list highlights
the need for \cpv{caution} when dealing with older resources.
While the congruence of the three methods is partial,
the \gls{TS} genes show distinct expression profiles across tissues
that are rather consistent through the different studies.\mybr\

I have also explored the congruence of several other genes categories
across the studies following a classification
inspired by Uhlén \etal\ publications~\mycite{Uhlen2014,Uhlen2015,Uhlen:2016}.
These gene categories are based on the level and breadth of expression
(\enquote{Not detected}, \enquote{Not expressed at $1$ \FPKM},
\enquote{Ubiquitous low expression ($< 10$ \FPKM)},
\enquote{Ubiquitous high expression ($≥ 10$ \FPKM)},
\enquote{Mixed low expression (when expressed, $< 10$ \FPKM)},
\enquote{Mixed high expression (when expressed, $≥ 10$ \FPKM)},
\enquote{Group Enhanced},
\enquote{Tissue Enhanced},
and \enquote{Tissue Enriched}).\mybr\

Finally, I have compiled all the genes showing a consistent pattern of expression
through the meta-analyses across the studies into curated sets.\mybr\

Since I started this project,
other research groups have published similar studies.
However, at the time of writing this thesis,
all the other studies (including the aforementioned ones)
were still based on the human genome build \hg{37} (or hg19),
while I am using the more recent \hg{38} one.
These studies either have different focus, aims,
approaches or more limited scopes.
Below, I outline how my work expands or completes theirs.\mybr\
\begin{itemize}[topsep=0pt,nosep]
%Feb 2016 -Uhlén
\item\hspace{-1mm}\fullcite{Uhlen:2016} This review
presents the results of \citet{Yu2015-uh} and \citet{Danielsson2015-cn}
discussed in the following points.
It also compares data released by the \gls{GTEx} consortium~\mycite{Bahcall2015-mh,GTEx_Consortium2015-si,Gibson2015-wh}
to its authors' own dataset~(\uhlen\ data~\mycite{Uhlen2014,Uhlen2015}).
As many findings from other studies (that I discuss hereafter) are reviewed,
the comparison of the two studies is
limited to the examination of
the proportion of genes in each category of a simplified classification
for nineteen tissues.
We reach the same conclusions:
overall, there are significant overlaps across the datasets for each of the
categories they have considered in their study, \ie\ \enquote{Expressed in all},
\enquote{Not detected}, \enquote{Tissue enriched}, \enquote{Group enriched},
\enquote{Enhanced} and \enquote{Mixed}.\mybr\
%December2015 -Burge
\item\hspace{-1mm}\fullcite{Sudmant2015-zt}.
The authors focus on interspecies, intertissue and interstudy comparisons.
The major issue of the study is its use of \gls{TMM} as a gene expression unit.
\gls{TMM} normalisation has the assumption that
genes have a stable expression across conditions while the different analyses
I have presented indicate that many \pcgs\ expression profiles show tissue
specificity while they have a stable expression across the studies.
They also limit their scope to very specific orthologs
since they explore \Rnaseq\ expression data across species, tissues and studies.
They confirm that with \Rnaseq\ expression profiling,
the interstudy technical variation is generally lower than
the intrastudy biological one, \ie\
interstudy homologous tissues of the same species are usually
closer in similarity than different tissues of the same species
(or matched tissues of different species) of the same study.
They found that interstudy comparisons are more variable for \species{human}
than other species.
They finally note that this kind of meta-analysis is dependent on
the choice of tissues to be studied.\mybr\
%June2015 - Yu
\item\hspace{-1mm}\fullcite{Yu2015-uh}, integrate \uhlen\ \etal\ data~\mycite{Uhlen2014}
with \gls{CAGE} peak expression data from the \gls{FANTOM5}
consortium~\mycite{FANTOM5-cage}.
Overall, their analyses are very similar to the ones I have presented in this
chapter.
We also reach similar conclusions as well:
\begin{itemize}[nosep,topsep=0pt]
\item Overall gene expression is comparable through their two datasets.
\item Tissue expression signatures are independent of the data set (and profiling method).\mybr\
\item Global comparison of ubiquitously expressed and \gls{TS} genes are comparable
    across the two studies
\item They compare the two datasets at gene levels because of the lack of accuracy
    of the current \Rnaseq\ protocols and algorithms and focus on the \pcgs\
    as the level of agreement between the two studies is low
    (which they attribute to the polyA-selected protocol of \uhlen\ \etal\ data).\mybr\
\end{itemize}
Besides the choice of the original studies,
the few differences are (1) the version of the annotation
(they use the previous human genome build (\hg{37})) and
(2) they apply more simplified classification for ubiquitous and \gls{TS} genes.
In addition, they are also more lenient to assess the congruence
(\eg\ expressed in all tissues
in one dataset and 95\% of the tissues in the other datasets is considered as
concordant).
Strikingly, they also found a discrepancy with \salivary\ compared to the other tissues
which I have noticed in this study\footnote{\salivary\
is the only tissue for which \uhlen\ and \gtex\ show a Pearson correlation coefficient
$r<0.65$.}.\mybr\
%March2015
\item\hspace{-1mm}\fullcite{Danielsson2015-cn}, covers three different tissues
(\brain{}\footnote{Either \cortex\ or \hypothalamus}, \heart, \kidney)
extracted from five projects (\cite{Burge,VTpaper,Uhlen2015,Krupp2012} and \ibm).
Their study is limited to
the comparison of precomputed data (from the original laboratories)
versus uniformly reprocessed data (by themselves).
They are exploring experimental variation factors and possible correction strategies.
They reveal that original precomputed data have considerable study-specific biases.
Their results on interstudy tissue similarities are superficial.
One of their most \enquote{fine-grained} results is the very low number of shared
genes amongst the hundred most expressed genes
for the three tissues across their five datasets.\mybr\
%June2015 - SciRep-Zhu
%\item\hspace{-1mm}\fullcite{Zhu2016-xo}
%June2015 -Santos
\item\hspace{-1mm}\fullcite{Santos2015-rj}, focuses on the congruence of gene---tissue association
through different types of expression data (transcriptome and proteome)
and resources.
They report that most genes are either expressed in every
considered tissue or in small subsets in their constructed working dataset.
They also find that tissue specificity trends are globally similar,
even though there are many differences in the identified genes set across studies.
%even if there are many discrepancies for the genes identified within a same tissue
%across the original datasets.
They have integrated together five tissues
(\heart, \kidney, \liver, \nervous\ and \intestine)
from five transcriptome datasets that they use \enquote{as-is}\footnote{%
Even in the follow-up paper,
where they reprocess all the \Rnaseq\ data
for \species{mouse}, \species{rat}, \species{pig},
\citet{Palasca2018-fh} do not mention any improvement for the \species{human} \Rnaseq\
data (either in the results or methods and data).},
before refining a complementary set that comprises only the three highest-quality
datasets: \WebFoCi{UniGene database}{https://www.ncbi.nlm.nih.gov/unigene}%
{Wheeler2003-dv,unigeneNcbi}, \uhlen\ \etal\ data (HPA \Rnaseq)~\mycite{Uhlen2014,Uhlen2015}
and \castle\ \etal\ (\Rnaseq\ atlas)~\mycite{Krupp2012} for which they provide
association data for 14,722 genes.
They forsake the direct quantitative exploration and comparison of gene expression
between the studies,
but examine the tissue association enrichment
through the gene expression fold change (FC) for a qualitative cross-study.
The main issue with their transcriptomic study is their
assumption that higher expression\footnote{\FPKM\ values are directly
used as score for true presence and selectivity to a tissue} means
more robust gene---tissue association
which may often (but wrongly) be translated to a greater tissue-specificity.\mybr\
%Sep2017
\item\hspace{-1mm}\fullcite{Wang2017-hc}, have used
subsets of \gtex\ and \tcga\ raw data that they have quantified \mRNAs\ at
transcript levels with \hg{37} (hg19).
They have corrected for the study effect with \softCi{ComBat}{Johnson2007-xh}
and have released the normalised data to the community.
Note that \egxa\ provides more recent versions of the \gtex\ and \tcga\
(as a part of the \pcawg\ project\footnote{\pcawg\ project
analyses conjointly all available kind of research data related to cancer}) data.\mybr\
%Sept.2014
\item\hspace{-1mm}\fullcite{seqcmaqc}, have found that relative expression measurements
by \Rnaseq\ are accurate and reproducible across sites.
The authors also showed that the overlap of identified and characterised genes
is imperfect ($91$\%)
even when the design includes
the same two well-characterised reference RNA samples across all sites.
Also,
this specific design prevents inferring how the biological signal
may compare to the individual variations and the possible noise introduced
by collection, storage and extraction protocols.\mybr\
\item Several papers (\cite{Khang2015-qt,ruvseqComQN,Rau2014-va})
    explore the reliability of \Rnaseq\ in the context of
    \gls{DGEA} which is outside the scope of this thesis.\mybr\
\item\cite{Zhuo2016-qi} explore the stable expressed genes across multiple
    (24) \Rnaseq\ studies for \species{Arabidopsis}
    which is why I will not discuss it further.\mybr\
\end{itemize}


In summary, while the expression levels are hard to translate directly
from one study to another~\mycite{Yu2015-uh,Santos2015-rj},
many facts have been highlighted in this thesis with a subset of them confirmed
by the studies mentioned before:\mybr\ %

\begin{itemize}[nosep,topsep=0pt]
        \item Tissues are clustering preferably with corresponding
            (or closely biologically related) tissues even across studies
            rather than clustering with different tissues from the same study
            (\ie\ biological signal $>>>$ technical noise due to \Rnaseq\ protocols).
        \item More recent transcriptome studies are more congruent than previous ones.
        \item \testis\ presents the highest number of \gls{TS} \pcgs\
            (see \Cref{fig:UniqExprPC1}).
            It also presents the most variety of expressed \pcgs\ ($≥1$ \FPKM)
            \castle, \brawand, \ibm\ and \uhlen\ studies.
            This extends to \gtex\ study if all detected genes (\ie\ above 0 \FPKM)
            are considered.
        \item \liver\ has the most robust \gls{TS} \pcgs.
            It may be explained either by a robuster gene expression,
            its greater homogeneity than most tissues,
            or a greater knowledge and better annotation
            than the other tissues.
        \item Most genes are ubiquitously expressed while
          a small proportion are expressed in a very limited set of tissues.
        \item Well-differentiated tissues have specific expression profiles
            that allow using processed data \enquote{as-is} for rough comparisons
            such as sample swap checks
            or quality controls, \eg\ \salivary\ in \uhlen\ \etal\ data which
            presents low correlation with
            \gtex\ data (see \Cref{fig:SamedistribPearsCorr},
            p.~\pageref{fig:SamedistribPearsCorr}) and with
            FANTOM5 data~\mycite{Yu2015-uh}.
        \item Annotations have an essential effect on the final results.
              Thus, whenever possible, we ought to keep the resources up-to-date.
\end{itemize}


\vspace{2.5mm}
Unsurprisingly,
updating the human genome version from \hg{37} to \hg{38}
for the reconstruction
step\footnote{See \Cref{subsec:reconstruction}: \nameref{subsec:reconstruction}}
enhances the results significantly.\mybr\

Besides, as my analyses were incorporating more studies,
the results were supporting more similarity in the gene expression levels across
the tissues and studies.
Indeed, as I focus my analyses on the common set of genes across the studies,
I remove the most interstudy variant genes
(\ie\ that are probably more sensitive to technical factors), and
I bias the analyses towards the genes
for which \Rnaseq\ is more robust to quantify their expression profiles.
Hence, it may be interesting to relax the filters by including genes
that are found in any two or more datasets as a follow-up study.\mybr\

\section{Conclusion}

The meta-analyses show that \Rnaseq\ captures
a strong biological signal for tissue gene expression
despite any noise created by batch effect or technical variations.\mybr\

Tissues reference expression profiles (\treps) are well correlated
across independent studies and are the sum of the overall genes contributions.\mybr\

While highest expressed genes fail to show significant correlation between
the different studies,
the analyses show that most
gene expression profiles are comparable from one study to another.
The \emph{gene centred} heatmap (\Cref{fig:egxaSample}) available now
in \WebFoCi{\egxa}{https://www.ebi.ac.uk/gxa/}{EBIgxa}
and its corresponding widget
are a direct translation of this observation.\mybr\

To assist further research,
I provide extensive curated and consolidated gene sets
for the different categories reviewed in the above analyses,
\ie\ for the \gls{TS} genes,
and the categories derived from Uhlén publications:
\enquote{Not detected}, \enquote{Not expressed at $1$ \FPKM},
\enquote{Ubiquitous low expression ($< 10$ \FPKM)},
\enquote{Ubiquitous high expression ($≥ 10$ \FPKM)},
\enquote{Mixed low expression (when expressed, $< 10$ \FPKM)},
\enquote{Mixed high expression (when expressed, $≥ 10$ \FPKM)},
\enquote{Group Enhanced},
\enquote{Tissue Enhanced},
and \enquote{Tissue Enriched}.\mybr\

\begin{sidewaysfigure}
    \includegraphics[scale=0.45]{transcriptomics/egxaSample.png}\centering
    \caption[Example of EBI gene expression atlas gene centric heatmap]%
    {\label{fig:egxaSample}\textbf{Example of EBI gene expression atlas gene centric heatmap}.
    This heatmap shows the relative expression of the Albumine (ENSG00000163631)
    across the tissues and studies.
    Note that the expression is calculated within each study library before
    being aggregated by identical condition or tissue.}
\end{sidewaysfigure}

There is a need for more multi-tissue studies
with biological replicates to refine and complete the above findings
for other tissues and extend them to the transcript isoform level.\mybr\

New strategies, notably normalisation methods, have to be also developed
to allow the easy reuse of uniformly processed and quantified data by
the community.
Ideally, the final aim should be to provide a general human transcriptome build
as it already exists for the genome.
Finally, as long as the annotations are redefined and refined,
it~also~means~that~periodic~resources~reprocessing~may~be~inevitable.\mybr\


