\chapter{Expression in normal human tissues across RNA-Seq studies}
\label{ch:Transcriptomics}

In the perspective of paving the way towards
a generalised baseline expression reference for the normal human,
in this chapter, I assess the similarity
of the tissues sourced from different \Rnaseq\ studies and
the general profiles of their expressed genes.

All the work presented in this chapter was performed by myself under the
supervision of \alvis.
I received invaluable advise and help from my discussions with \nuno.
I also received general feedback and comments from \mar, \johan, \sarah, \gos\
and \wolfgang.

When I started this project in 2013,
little was then known on either the robustness or
the shortcomings and pitfalls of \Rnaseq\ and
its related processing.
Since then, several studies were published assessing \Rnaseq.
A few present a closely related scope to my own investigations, thus
whenever relevant,
I introduce and discuss my results in relation to the published ones.

\derivativeWork{}
\begin{itemize}[topsep=0pt,nosep]
    \item \fullcite{EBIgxa}
    \item (poster) ECCB 2014 --- A feasibility study:
        Integration of independent RNAseq datasets
    \item (invited talk) GM$^2$ 2013 --- Baseline Gene expression Atlas
    \item (flash talk) CSAMA 2013 --- How quantitative is RNA-seq?
\end{itemize}
\clearpage

\section{Working sets}

While many approaches exist,
I usually consider the most conservative routes.
Thus, I first identified the identical core of
explored tissues and expressed genes across the studies.
Then, from this base, I created working sets
that are less ambiguous for the following meta-analyses.

Through this chapter, I use two working sets:
\begin{itemize}[topsep=0pt,nosep]
    \item \setOne: 4 tissues --- 12,268 genes across the 5 \Rnaseq\ studies, and
    \item \setTwo: 23 tissues --- 17,551 genes across 2 of these studies.
\end{itemize}

The following \cref{subsec:transtissueOverlap,subsec:transGeneOverlap}
illustrate the construction of these sets.

\subsection{Tissue overlaps across the available normal human RNA-Seq studies}%
\label{subsec:transtissueOverlap}

\Cref{fig:VennStudiesT} presents the five datasets overlapping tissues.
Notice that they all share at least four tissues:
\heart, \kidney, \liver\ and \testis.
This 4-tissue set is the base of one working set.

\begin{figure}[h]%[!htbp]
\includegraphics[scale=0.50]{transcriptomics/TransVennTissue.pdf}\centering
\caption[Distribution of unique and shared tissues between the
transcriptomic datasets]
{\label{fig:VennStudiesT}\textbf{Distribution of unique and shared tissues
between the transcriptomic datasets.} The 5 datasets share 4
common tissues: \heart, \kidney, \liver\ and \testis.
The most prominent overlap of tissues (23) is between \uhlen\ and \gtex.
These two sets of tissues are the primary focus of the transcriptomic part of the
study.}
\end{figure}

Also, observe that the greatest number of shared tissues is
between the two most recent studies:
\uhlen\ and \gtex.
They constitute the base of a second 23-tissue working set.
This set includes
\Adipose, \Adrenal, \Bladder, \Cortex, \hcolon, \Esophagus,
\Fallopian, \heart, \kidney, \liver, \lung, \Ovary, \Pancreas, \Prostate,
\salivary, \skeletal, \skin, \intestine, \spleen, \stomach, \testis,
\thyroid\ and \uterus.

\subsection{Common measured genes for each of the common tissues sets}%
\label{subsec:transGeneOverlap}
As shown in \Cref{tab:Trans5DF},
many of the transcriptomic datasets I use have been produced through
polyA-selected library protocols.
Hence, most of the analyses are limiting to genes with a biotype annotated as
\emph{\pc}
to avoid unnecessary biases\footnote{See
\Cref{subsec:protcodingOnly}: \nameref{subsec:protcodingOnly}.}.

\begin{figure}[!htpb]
    \includegraphics[scale=0.50]{transcriptomics/PcodingGenesExpressed1_4tissues.pdf}\centering
    \caption[Unique and shared \pcgs\ expressed
    in the 4 common tissues (≥1 \FPKM)]{\label{fig:ExpGenePcoding1}\textbf{Unique
    and shared \pcgs\ expressed ≥ 1 \FPKM\ in the 4 common tissues
    across the 5 studies.}}
\end{figure}

The Venn diagram presented in \Cref{fig:ExpGenePcoding1} only includes protein
genes that are observed\footnote{See
\Cref{sec:ExpressedOrNot}: \nameref{sec:ExpressedOrNot}.}
at least once at 1 \FPKM\ for one of the four shared tissues.
Plainly, the bulk of genes expressed at this threshold is common
across the five datasets.
Indeed, while each study presents a tiny portion of genes
that are unique,
overall most genes are detected in two or more studies.
The most considerable contingent of shared genes is between \uhlen\ and \gtex.

\begin{figure}[h]
    \includegraphics[scale=0.45]{transcriptomics/vennTissue23_1protcodgenes.pdf}\centering
    \caption[Unique and shared \pcgs\ expressed
    in the 23 common tissues (≥1 \FPKM)]%
    {\label{fig:ExpGenePcoding1_t23}\textbf{Unique
    and shared \pcgs\ expressed ≥ 1 FPKM in the 23 common tissues
    of \uhlen\ and \gtex\ studies.}}
\end{figure}

In this respect, \Cref{fig:ExpGenePcoding1_t23} presents a similar Venn diagram
while focusing on the twenty-three shared tissue set between \uhlen\ and \gtex.
The number of unique genes to each study is negligible compared to the bulk:
it represents less than 0.03\% of the measured genes in each of the studies.
\begin{comment}
    Gtex:   462/17551 hence 0.02632329\%
    Uhlen:  281/17551 hence 0.01601048\%
\end{comment}

\NB\ None of the subgroups of uncommonly shared genes presents any functional
annotation enrichment.

\section{Prevalence of biological signal over technical variabilities at
tissue level}
\label{sec:Trans_ReproExpresTissue}

As I show in \Cref{ch:expression}, within the same study,
the expression levels of identical tissue samples are highly correlated and
allow to group the samples based on their biological source.
In fact, clustering the samples across the studies offers a quick assessment of
the underlying causes of the expression levels measurements.
A clustering by study means that the technical variabilities are stronger
than any biological expression signature.
On the other hand,
an interstudy sample clustering by tissue implies that \Rnaseq\ measurements
are less prone to batch effects and more robust than measurements by
microarray assays~\mycite{Taminau2014-hr,Walsh2015-nf}.

\Cref{fig:noMitoNoRep4T} and \Cref{fig:noMitoNoRep23T}
are the respective heatmaps of the hierarchical clustering
of the \treps{}\footnote{See \Cref{subsec:averagedTissue}:
\nameref{subsec:averagedTissue}.}
for the four shared tissues across the five datasets and the
twenty-three shared tissues between \uhlen\ and \gtex\ studies.
The heatmaps are based on the clustering of the \treps\ based on
their Pearson correlation coefficients with the Ward's method
of the \pcgs\ (≥ 1 \FPKM, and excluding
the mitochondrial genes).

The clustering plainly highlights a greater similarity of the \treps\
due to their biological origins over any possible
similarity due to technical variations of library preparation or processing.
Indeed, each cluster concurs to a tissue in \Cref{fig:noMitoNoRep4T}.
While we may object that
the very different gene expression (and levels) in
\Heart, \Kidney, \Liver\ and \Testis,
may drive this result,
\Cref{fig:noMitoNoRep23T} confirms that the biological origin of the tissues
is the dominant criterion for the clustering of the \treps.
Even if we observe a few mixture of \treps,
the majority of them cluster by tissue.
Moreover, in many cases, the mixture is due to close biologically related tissues,
\eg\ \fallopian\ and \Ovary, \salivary\
with \Esophagus\ or \Stomach\ \treps.
\Cref{fig:noMitoRep4T} and \Cref{fig:noMitoRep23T}
present heatmaps for the same tissues and studies,
but where every available sample is included\footnote{I.e.\
there may be several samples for the same tissue in each study.}.
These two supplementary figures also support that
the biological signal is in overall stronger than the technical variations.

\begin{figure}[!htpb]
    \includegraphics[scale=0.85]{transcriptomics/heatmap4TnoMitonoRep_1.pdf}\centering
    \caption[Heatmap of the 4 common tissues across the 5 studies]%
    {\label{fig:noMitoNoRep4T}\textbf{Heatmap of the 4 common tissues
    across the 5 studies.}\\All \pcgs\ (except the mitochondrial
    ones) at least expressed at 1 \FPKM\ are included.\\We observe that all the
    different \treps\ cluster by tissue (instead, for example, of studies)
    of origin.}
\end{figure}

\begin{figure}[!htpb]
    \includegraphics[scale=0.85]{transcriptomics/heatmap23TnoMitonoRep_1.pdf}\centering
    \caption[Heatmap of 23 common tissues between Uhlén and GTEx studies]%
    {\label{fig:noMitoNoRep23T}%
    \textbf{Heatmap of 23 common tissues between Uhlén and GTEx studies.}\\
    All \pcgs\ (≥ 1 FPKM with the exclusion of the mitochondrial
    genes) are included.\\Most \treps\ cluster by tissues but a few exception.
    Indeed, we observe a mixture of the \tissue{Fallopian tube}
    and \tissue{Ovary} \treps.
    Besides, \tissue{Salivary gland} \treps\ may be more correlated to
    \tissue{Esophagus} or \tissue{Stomach} regarding the original study.
    Only the \tissue{Bladder} \treps\ seem to cluster randomly with the others.
    However, these \treps\ are in singleton groups.}
\end{figure}

\Cref{fig:SamedistribPearsCorr} shows the distribution of the Pearson correlation
coefficients for the pairs of identical tissue \treps\
sourced from the different studies
for both of the working datasets.

Most of the Pearson correlations are above 0.5\footnote{Indeed, apart from the
correlation between the \Testis\ \treps\ of \castle\ and \vt\ (0.42)
for the 4-tissues working set and
\Salivary\ \treps\ of \uhlen\ and \gtex\ (0.2)
for the 23-tissues working set.}
even with the lack of any batch effect correction.
The median correlation for the four common tissues across the five datasets is
about 0.7 and 0.84 for the twenty-three tissues between \uhlen\ and \gtex\ studies.
Results are even better with Spearman Correlation:
the averages are respectively 0.49 for the 4-tissues sets
and 0.9 for the 23-tissues set and
the median correlations are 0.88 and 0.93.

Thus, both the Pearson\footnote{Despite one major outlier in the second
working set (\tissue{Salivary gland} --- Pearson correlation: 0.2)} and the
Spearman correlation coefficients for the more exhaustive 23-tissues working set
comprising the two most recent studies,
are higher than the observed correlation for the 4-tissues working set.
Three main reasons may explain this situation:
\begin{itemize}[topsep=0pt,nosep]
    \item Besides of using paired-end sequencing,
        the library preparation protocols were better established
        for these two studies;
    \item The instrument used for the sequencing were
        from the same series (HiSeq 2000 and HiSeq 2500); and
    \item These studies present a higher number of replicates per tissue.
\end{itemize}

\begin{figure}[!htpb]
    \includegraphics[scale=0.65]%
{transcriptomics/TransPearsonDistributionIdenticalOnly.pdf}\centering
\caption[Distribution of the correlation of same tissue pairs for the 4 and 23
tissues working sets.]{\label{fig:SamedistribPearsCorr}\textbf{Distribution
of the Pearson correlation of same tissues pairs for the 4 and the 23 tissues
working sets.} In general, the Pearson correlations are high even though we are
\emph{directly} comparing \treps\ from different studies.\\
The 23-tissues working set (\setTwo) presents a higher median correlation ($0.85$)
and narrower distribution for the pairs bulk than
the 4-tissues working set (\setOne) (median$ = 0.74$).
However, \setTwo\ displays one outlier with
a very low Pearson correlation ($0.2$: \salivary\ tissue.).
Sampling, biological or processing issues may just as well explain this outlier.}
\end{figure}

\Cref{fig:distribCorr} presents the different pairs of tissues across the
datasets in addition to the same tissue pairs.
The pairs comprising different tissues are very lowly correlated in general.

\NB\ In few cases, for the 23-tissues working set,
high correlations are also observed (see also \Cref{fig:noMitoNoRep23T}).
It is rather hard to decipher if this may not be due to an issue at the collection
or library preparation stage or because these tissues are biologically very close.

\section{Possible driving force of the closer intratissue rather intrastudy
similarity}

Since \Rnaseq\ allows distinguishing the shared biological origin
of most \treps\ across different studies,
the question then arises as to what is driving these strong interstudy
correlations.

As discussed in \Cref{ch:expression} (see \Cref{sec:ExpressedOrNot}:
\nameref{sec:ExpressedOrNot}),
correlation coefficients measure the dependence between (here) two tissue \treps\
and are subject to outliers and the skewness of the distributions.
Thus, the next intuitive step is to test
whether a particular subset of the genes may drive the correlation coefficients,
\ie\ are the highest, the most variable or another group of genes
the underlying reasons of the strong correlations.
Indeed, as I have excluded the \emph{undefined}\footnote{I.e.\
\emph{unobserved} --- See also
\Cref{subsec:ExpressedOrNot-undefined}: \nameref{subsec:ExpressedOrNot-undefined}}
genes from the analyses,
the high correlations have another rationale than spurious null values.

\subsection{Highest expressed genes}

The notion of highest expressed genes may be trivial,
but it is worth remembering that the normalisation methods are impacting
the final expression values.
Note that individual genes may have, or not,
the highest expression values for a given condition
depending on the normalisation methods
(see \Cref{subsub:norm}:~\nameref{subsub:norm})
Although, in the present case,
by processing the data with consistent methods and annotation
and by excluding selected sets of
genes\footnote{See \Cref{sec:bias_sources}:~\nameref{sec:bias_sources}}
I avoid many biases.

\NB\ If I had picked another normalisation method than \FPKM\footnote{See
\Cref{eq:rpkm-fx}},
the exclusion filter would impact a different set of genes.

To test the influence of highest expressed genes on correlation coefficients,
I chose to visualise the evolution of the correlation
in function of expression value thresholds.
Indeed, for \Cref{fig:CorHighExp4T} and \Cref{fig:CorHighExp23T},
I computed the correlations of genes expressed above a cut-off
for each pair of identical tissues within each of the two working datasets.
The cut-off is a range of possible (integer number) value of gene expression
(using 10 as a step).

In \Cref{fig:CorHighExp4T}, that shows the 4-tissues working set,
except for very few pairs, the highest correlations correspond to
the lowest cut-off of 1 \FPKM\@.
In fact, the same tissue \trep\ correlation coefficients
are increasing as the expression cut-off is lowered
(the calculations involve then more genes).
Here below, the few exceptions grouped by tissue:
\begin{eqlist}[\eqliststarinit\def\makelabel#1{\bfseries#1}\labelsep1em]
\item[Heart] \uhlen{}-\gtex\ pair
\item[Kidney] \uhlen{}-\gtex, \castle{}-\uhlen\ and \castle{}-\gtex\ pairs,
\item[Liver]  \vt{}-\ibm, \ibm{}-\uhlen\ and \ibm{}-\uhlen\ pairs;
\item[Testis] \ibm{}-\uhlen, \vt{}-\gtex, \vt{}-\uhlen\ and \uhlen{}-\gtex\ pairs
\end{eqlist}

In the context of the 23-tissues working set,
shown in \Cref{fig:CorHighExp23T},
many more tissue pairs present very high correlation for subsets of their highly
expressed genes, \ie\ \skeletal, \Thyroid, \Cortex, \Uterus, \Kidney.
Unfortunately, these specific examples are insufficient
to derive any consensual threshold for future work.

\begin{figure}[!htpb]
    \includegraphics[scale=0.8]{transcriptomics/T23EvolHighExp23P-1.pdf}\centering
    \caption[Pearson correlation coefficient evolution based on the expression
    levels of the genes considered for each of the 23 common tissues]{%
\label{fig:CorHighExp23T}\textbf{Pearson correlation coefficient evolution based
on the expression levels of the genes considered
for each of the 23 common tissues between \uhlen\ and \gtex.}
Almost only the complete set of common expressed \pcgs\ of each tissue gives
the highest correlations.}
\end{figure}

\begin{sidewaysfigure}[htpb]
    \centering
    \begin{subfigure}[b]{0.50\textwidth}\centering
        \includegraphics[width=\textwidth]{transcriptomics/HeartEvolHighExp4P-1.pdf}
        \caption{Heart}\label{fig:CorHighExpHeart4T}
    \end{subfigure}%
~%
    \begin{subfigure}[b]{0.50\textwidth}\centering
        \includegraphics[width=\textwidth]{transcriptomics/KidneyEvolHighExp4P-1.pdf}
        \caption{Kidney}\label{fig:CorHighExpKidney4T}
    \end{subfigure}

    \begin{subfigure}[b]{0.50\textwidth}\centering
        \includegraphics[width=\textwidth]{transcriptomics/LiverEvolHighExp4P-1.pdf}
        \caption{Liver}\label{fig:CorHighExpLiver4T}
    \end{subfigure}%
~%
    \begin{subfigure}[b]{0.50\textwidth}\centering
        \includegraphics[width=\textwidth]{transcriptomics/TestisEvolHighExp4P-1.pdf}
        \caption{Testis}\label{fig:CorHighExpTestis4T}
    \end{subfigure}
    \caption[Pearson correlation coefficient evolution based on the expression
    levels of the genes considered for each of the 4 common tissues]{%
\label{fig:CorHighExp4T}\textbf{Pearson correlation coefficient evolution
    based on the expression levels of the genes considered for each of the 4
    common tissues across the 5 studies.}}
\end{sidewaysfigure}

Moreover, \kidney\ aside,
for any similar tissue pair of \setTwo,
considering every protein gene expressed at least at 1 \FPKM\ is usually
far better than selecting any highly expressed genes subset.

As the interstudy Spearman correlations of same tissue pairs
are higher than the Pearson ones,
I have tried to ponder for the possible difference in the expression magnitude
across the studies\footnote{Due to particular batch effects in each study}
while the highest expressed \pc\ may have similar ranks
for a given tissue.
Thus, I have explored the ratio evolution of the common most top expressed
genes across the studies to the number of genes considered.

\Cref{fig:highExpress4T} shows across the studies and for each tissue,
the ratios of the number of
common \pcgs\ for a given amount of highest expressed \pcgs\
to that very number.
Among these (cumulative) proportions,
a few present very high value for
a minimal subset of genes (below $10$ \FPKM\ across all the tissues)
which then drop dramatically to finally increase slowly
to reach the expected ratio of $1$ \FPKM{}.
\begin{comment}
as the \pcgs\ set across the studies is identical.
\end{comment}

\Cref{fig:highExpress23T} presents the same kind of ratios,
however, limited to \uhlen\ and \gtex\ solely.
Here as well, aside from the expected perfect ratio for the complete set of
\pcgs,
only a tiny subset of the highest expressed genes produce high rates of
highly expressed common genes to the number of considered ranked genes.
These results are quite unsurprising as they involve only two studies.
In addition to increasing the number of overlaps probabilistically,
these two studies are also the two most recent ones
and comprise a higher number of replicates per tissues;
thus the measures for each gene in each condition are likely more robust.

Comparing the calculated ratios between the real (colour) and
the randomly permuted data (grey)
on \Cref{fig:highExpress4T,fig:highExpress23T}
plainly show that across the assortment of studies,
there are common (biological) structures
that are not fortuitous.

While a study of the highly expressed \pcgs\ is interesting,
its ability on explaining the underlying reasons
of the strong interstudy tissue correlations seems limited at best
or even inadequate.
Thus, I have considered other candidates
such as the most variable genes.

\subsection{Most variable genes}

Similar expression variation patterns across the tissues
within the independent datasets may explain the strong correlations.

There are several available estimators to describe
the variability of gene expression,
\eg\ the variance or \gls{cv}.

However,
I only report here the results based on the coefficients of variation
as the analyses involving the variance instead give
fewer insights into the correlation strength
between the tissues across the datasets.
Also, it is worth mentioning that the results presented here
improve considerably when the analyses also include \uhlen\ and the \gtex\ studies.
\begin{comment}
Indeed, preliminary results (only based on \castle, \vt\ and \ibm\ studies)
were far worse.
\end{comment}
Unsurprisingly, updating the human genome version from \hg{37} to \hg{38}
for the reconstruction
step\footnote{See \Cref{subsec:reconstruction}: \nameref{subsec:reconstruction}}
also enhances significantly the results.

The coefficient of variation (\gls{cv}) allows assessing
the dispersion of the gene expression values
across the tissues within each dataset.
As it contextualises the values to the mean,
it is a more straightforward estimator to interpret than
the standard variance,
in particular for interstudy comparisons.

Visualising the \gls{cv} distribution of
gene expression for the working set \setOne\ (see \Cref{fig:HistCV4T})
allows determining whether they are similar across the five transcriptomic studies
or that inferring conclusions requires more cautions.
As the \cvs\ present the same pattern across the five studies,
it suggests that extra steps are non-compulsory here.

\begin{figure}[!htpb]
    \captionsetup{singlelinecheck=off}
    \includegraphics[scale=0.75]{transcriptomics/distributionCV_4commonTissues.pdf}%
    \centering
    \caption[Coefficients of variation across the 5 studies for the set of common
expressed genes and tissues]{\label{fig:HistCV4T}\textbf{Distribution of the
\cvs\ (cv) across \setOne: common set of expressed \pcgs\
across the  4 common tissues:
\{\Heart, \Kidney, \Liver, \Testis\}
across the 5 studies.}\\
The coefficients of variation (\gls{cv}) of the \pcgs\ (12,268) of the four tissues
present the same bimodal distribution profile across the five studies.
\\These profiles present two pics: at $0.5$ and $2$.\\
After more investigation, it appears that
the genes with a \gls{cv} lesser than or equal to $0.5$ have
a similar expression profile to a left-truncated version of
the complete gene set ones (due to the $1$ \FPKM\ cut-off)
as in \Cref{fig:distribPlot}.
On the other hand, the \pcgs\ with a coefficient of variation
equal to or greater than $1.5$ have two kinds of distinct profiles:
{\small
\begin{itemize}[topsep=0pt,nosep,leftmargin=95pt,listparindent=5pt]
    \item The gene expression is low across the four tissues and
        it is above the cut-off of $1$ \FPKM\ only once; or
    \item The gene expression is specifically high for one single tissue
        in regards to the three others.
\end{itemize}
}}
\end{figure}

A simple approach to test if the commonly most variable genes
are the driving force of the correlation is
to study the difference of their correlation with
the remaining set of considered \pcgs.
If the most variable \pcgs\ are driving the correlation
then their group has to present higher correlations than the second one.
\Cref{fig:test_mostvaribleVSevery} summarises this analysis.

\Cref{fig:test_mostvaribleVSevery} shows the Pearson correlation distribution
for these two groups of \pcgs\ in \setOne.
After ranking the \pcgs\ in decreasing order of their \cv\ within each study,
I select the intersection of genes in the first quarter across them
to create the first group ($S_{most~Variable}$).
The remaining genes are constituting the second group ($S_{remaining}$ and
$S_{remaining} =$ \setOne{}$- S_{most~Variable}$).
For both of these gene groups, $S_{most~Variable}$ and $S_{remaining}$,
I calculate the coefficients of correlation for all possible tissue couples.

One-sided \Welchttest\footnote{See \Cref{mini:ttest}.}
at 95\% of confidence,
allows discarding the null hypothesis $H_0$.
The mean of the %(Pearson and Spearman)
correlation coefficients
calculated with the common most variable \pcgs\ are significantly greater than
the ones calculated with the remaining set of genes in \setOne\footnote{Though,
a two-sided \Welchttest\ compels to accept the $H_0$ hypothesis:\\
the two populations of coefficients correlation are not significantly different
(\pvalue{= 0.06594}).}, \eg\ for the Pearson correlations:
$mean_{most~Variable}=0.76$ and $mean_{remaining}=0.68$
(\pvalue{= 0.03295}).
Thus, the common \pcgs\ with the highest \cv\
discriminate better between the identical and different tissues.

\begin{figure}[!htpb]
    \includegraphics[scale=0.75]%
    {transcriptomics/TransPearsonDistributionIdenticalDifferentHighestCVgenes.pdf}%
    \centering
    \caption[Comparison between the most variable genes with all the other ones]%
    {\label{fig:test_mostvaribleVSevery}\textbf{Comparison between
    the most variable \pcgs\ with the remaining genes in \setOne\ set.}
    The clustering is based on Pearson correlation between the different
    \treps\ of the various studies.
    Either for the most variables genes than for the other genes,
    the correlations between identical interstudy tissues are greater
    than any correlation between different (even intrastudy) tissues.}
\end{figure}

However, for both groups,
$S_{most~Variable}$ and $S_{remaining}$,
the \treps\ across the studies cluster mostly by tissue rather than
original study.
See \Cref{fig:heatmapMost25pVariable,fig:ReverseheatmapMost25pVariable}.\footnote{They
respectively present the heatmaps (with clusterings) of the tissues
based on their Spearman correlation for the most variable (\cv) \pcgs\
and all other remaining genes of \setOne.}
In fact, the only exception is that
\castle\ tissues cluster primarily by study
once I exclude the most variable genes.

\begin{figure}[!htpb]
    \includegraphics[scale=0.9]{transcriptomics/ConceptOverlap.png}\centering
    \caption[Overview for the comparison of the genes across the five
    studies based on a ranked descriptor 5 studies]{\label{fig:overlapConcept}%
    \textbf{Overview for the comparison of the genes across the five
    studies based on a ranked descriptor.}
    The first step applies individually to each of the studies
    within the working dataset (\ie\ here \setOne).
    It consists in extracting a single value per gene
    (\eg\ a statistic or any other quantitative descriptor)
    either for the entire dataset (referred thereafter as \emph{D-approach}) or
    for each tissue in each dataset (referred as \emph{T-approach}).
    The next steps include
    computing (cumulatively) the intersection size number for each rank
    and plotting this number divided by the rank
    in function of the number of considered genes (\ie\ rank).}
\end{figure}

\begin{figure}[!htpb]
    \includegraphics[scale=0.9]{transcriptomics/CVevolCumul5DF_lineLegend.pdf}\centering
    \caption[Evolution of the intersection size of \setOne\ genes (ranked by cv)]%
    {\label{fig:cvEvol5DF}\textbf{Evolution of the intersection size
    of \setOne\ genes (based on their coefficient of variation rank
    in each of the five studies).}
    There is an initial strong growth (a),
    followed by a short plateau (b)
    before a slight decrease (c)
    which then settles a plateau (d).
    Eventually, the ratio increases slowly again
    until reaching the expected ratio of $1$ once all the genes from \setOne\
    are included (e).
    The first quarter of the genes covers (a), (b), (c)
    and a tiny part of (d).
    Apart from (a),
    the overlap of shared genes between the 5 datasets when ranked on their
    coefficient of variation is above 70\%.
    The sigmoid curve (dashed line) is based on randomised data
    where permutations also destroy the original order of the genes.
    Notice the significant dissimilarity between the real and the randomised data.}
\end{figure}

To avoid any oversight due to the arbitrarily chosen number (first quarter)
of the most variable genes,
I have also studied the evolution of the intersection size of the common genes
across the five studies
in function of the number of \setOne\ genes that I consider.
\Cref{fig:overlapConcept} illustrates my general approach and
\Cref{fig:cvEvol5DF} presents the result.
Many of the most variable genes are commonly present in the top tier of the
five studies, though they have different individual rank.
Using the first quarter of the most variables genes as a cut-off appears
to be an acceptable pick among others.

\begin{comment}
If the previous analysis was to include
only the first 1,250 most variable genes
the results of \Cref{fig:heatmapMost25pVariable} may be even greater.
However, the dissimilarities highlighted by \Cref{fig:ReverseheatmapMost25pVariable}
would also be greater.
\end{comment}

In conclusion, while the most variable genes have a significant influence
on the strong and biologically meaningful correlations,
there are incontestably other contributing genes.

Genes that undeniably have an expression profile
that varies in function of the tissue,
and thus impact positively the correlations,
are the tissue specific genes.

\subsection{Tissue specific genes}\label{sub:TisSpeGene}

Another good candidate to drive higher correlations
between identical interstudy \treps\ than intrastudy ones are
the tissue specific genes.

There are different definitions in the literature.
There are also database as \eg\ \gls{TIGER}.

As a matter of fact, the expression patterns of many of
the most variable genes across the different tissues
are globally uniform aside one or two tissues
where the expression is higher (see \Cref{fig:expressionMostvariableG}).

\subsubsection{Use of prior knowledge: TiGER database}\label{subsub:Tiger}

\gls{TIGER} is a database that reports tissue-specific genes for thirty tissues
(based \glspl{EST} experiments).

After retrieving the list of genes from the thirty types of reported tissue, I have mapped the Refseq ID to Ensembl gene ID (Ensembl 73).
I filtered out all the genes reported more than one tissue. The following heatmap (Figure 7) shows a sample of these tissue specific genes reported by TIGER. We can observe different subsets of genes: the ones which are confirmed, the ones that are infirmed (i.e. expressed everywhere) and the ones that are attributed to another tissue.


One database in particular, \gls{TIGER},
has collate tissues-specific genes (based \glspl{EST} experiments)
for thirty independent tissues\footnote{See \Cref{sec:supplTiger}: \nameref{sec:supplTiger}
for the complete list.}
After I retrieved all thirty lists,
I have translated the gene lists from \gls{Refseq} to \gls{Ensembl} (\hg{38},v. 76).
Then, I removed all duplicates due to the translation and I also filtered out
all the genes identifiers that I found in more than one tissue.
Thus, for each tissue,
I have a list of identifiers that are specific to that tissue only.


\subsubsection{Hampel method}\label{subsub:Hampel}
To expand on the most variable observation, I use the Hampel method.
Explain a bit the Hampel method and how I have used as a test.

This method is robust for detecting outliers and has been run on the original datasets.
Once again, this part is not entirely done: I need to aggregate the results per condition within dataset before comparison with the other datasets.



\begin{figure}[!htpb]
    \includegraphics[scale=1]{transcriptomics/hampel5DF4Tissues.pdf}\centering
    \caption[Expression of the genes picked with Hampel method]{\label{fig:hampelExp}%
    \textbf{Expression of the genes picked consistently with the Hampel method
    in each study solely in one tissue.}}
\end{figure}



\subsubsection{Fold change method}\label{subsub:TisSpeGeneMethodPerso}

\begin{figure}[!htpb]
    \includegraphics[scale=1]{transcriptomics/mostSpe4TP.pdf}\centering
    \caption[Cumulative shared set of genes sorted by their specificity in each
    tissue across the 5 datasets]{\label{fig:mostSpe4T}\textbf{Cumulative shared
    set of genes sorted by their decreasing order of specificity in each tissue
    across the 5 datasets.} Results are better than for the highest expressed or
    the most variable genes.}
\end{figure}

Thus, the tissue specific genes are also contributing to provide a stronger
biological signal over the technological noise.

\subsection{Uhlén categories}\label{sub:UhlenGeneCat}

Finally, best may be to use different categories to describe the data as have
done Uhlén and co for example. Ca change de répartir les gènes que en exprimé/pas exprimé
et en niveau d'expression.

They change the definition of their categorisation
between their two related papers~\mycite{Uhlen2014} and~\mycite{Uhlen2015}.
The definition I am using is a based on their second paper,
but with some subdivisions they presented in the first one.

\pagestyle{plain}
\begin{landscape}
\begin{table}[]
\centering
\caption[Uhlén et al.\ gene categories]{\label{tab:UhlenCategoriesProtCoding}%
\textbf{Uhlén et al.\ gene categories}\\
\footnotesize{Apart the undetected genes and the ones expressed below 1 \FPKM,
a gene may be referenced in several categories.}}

%\begin{tabular}{@{}lllllllllll@{}}
\begin{tabular}{@{}ccccccccccc@{}}
\toprule
\multicolumn{2}{c}{\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\ens{76}
    \\($\sim$22,500 protein\\coding genes) \end{tabular}}} &
\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Not\\detected\end{tabular}} &
\multirow{3}{*}{\begin{tabular}[c]{@{}c@{}}Not expressed\\ at 1 \gls{FPKM}\\
    cut-off\end{tabular}} &
\multicolumn{2}{c}{Mixed expression} &
\multicolumn{2}{c}{Ubiquitous expression} &
\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Group \\Enhanced\end{tabular}} &
    \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Tissue\\ Enhanced\end{tabular}} &
        \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}\\Tissue\\ Enriched\end{tabular}} \\
    \cmidrule(lr){5-8}
\multicolumn{2}{c}{}
    &  &  &
    \begin{tabular}[c]{@{}c@{}}Low\\ (\textless\ 10 \gls{FPKM})\end{tabular} &
        \begin{tabular}[c]{@{}c@{}}High\\ (≥ 10 \gls{FPKM})\end{tabular} &
            \begin{tabular}[c]{@{}c@{}}Low\\ (\textless\ 10 \gls{FPKM})\end{tabular} &
    \begin{tabular}[c]{@{}c@{}}High\\ (≥ 10 FPKM)\end{tabular} &  &  &  \\
        \midrule
        \multicolumn{1}{c}{%
        \multirow{6}{*}{\rotatebox[origin=c]{90}{\parbox[c]{3cm}{\centering Whole
        dataset}}}} &
        Castle & 3,403 & 3,268 & 8,773 & 1,033  &
        1,399  & 634   & 11   & 3,664   & 1,975 \\
        \multicolumn{1}{c}{} & Brawand & 2,964 & 3,095 &
        8,034 & 1,788  & 1,760 & 958   & 0  &
        2,729  & 2,548 \\
        \multicolumn{1}{c}{} & IBM & 2,693 & 2,605  &
        7,325  & 1,406  & 1,135 & 858  & 322 &
        5,248  & 2,453  \\
        \multicolumn{1}{c}{} & Uhlén & 2,662 & 1,747 &
        5,769 & 1,053  & 456 & 406  & 2,511  &
        5,201  & 2,333  \\
        \multicolumn{1}{c}{} & GTEx & 2,197  & 1,886  &
        5,556  & 1,117 & 687  & 698  & 3,859  &
        4,356  & 1,919 \\ \cmidrule(l){2-11}
        \multicolumn{1}{c}{} & Consensus & 2,197 & 486  &
        1,749  & 221  & 33  & 161  & 0  & 677  &
        \begin{tabular}[c]{@{}c@{}}531 $[$518$]$\end{tabular} \\
            %\multicolumn{1}{c}{} & \footnotesize{without Gtex} &
            %\footnotesize{2,413} & \footnotesize{638} & \footnotesize{2,152} &
            %\footnotesize{286} & \footnotesize{63}  & \footnotesize{179} &
            %\footnotesize{0}  & \footnotesize{814} &  \footnotesize{587}  \\
            \midrule
\multirow{6}{*}{\rotatebox[origin=c]{90}{\parbox[c]{3.5cm}{\centering Common\\
4 tissues\\ Working datasets}}} &
Castle & 19,066 & 2,994 & 8,589 & 1,513 &
2,994 & 1094 & --- & --- & 2,185 \\
& Brawand & 19,505  & 2,962  & 8,626  & 2,228
& 2,962  & 1251 & --- & --- & 3,672  \\
& IBM & 19,776  & 2,989 & 8,534 & 1,954 &
2,989  & 1212  & --- & --- & 2,824  \\
& Uhlén & 19,807 & 2,917 & 8,367 & 2,227 &
2,917  & 1190  & --- & --- & 3,730  \\
& GTEx & 20,272 & 3,870 & 8,988  & 2,312 &
3,870  & 1427  & --- & --- & 3,554  \\
\cmidrule(l){2-11}
& Consensus & 1,973 & 550 & 3,351 & 649 &
550  & 439 & --- & --- & 1,412  \\
%& \footnotesize{without Gtex} & \footnotesize{2,413}  & \footnotesize{2,186}  &
%\footnotesize{3538}  & \footnotesize{639}  & \footnotesize{576}  &
%\footnotesize{439}  & \footnotesize{---} & \footnotesize{---} & \footnotesize{1,462}
\midrule
\multirow{3}{*}{\rotatebox[origin=c]{90}{\parbox[c]{1.7cm}{\centering Common\\ 23
tissues\\ Working datasets}}} & Uhlén & 2,662  & 1,970  &
6,160 & 1,135 & 594  & 427 & 1,285 &
5,776 & 2,518 \\
& GTEx & 2,197 & 2,258 & 6,966  & 1,540 &
1,822  & 997 & 1,048 & 5,496  & 2,460 \\
\cmidrule(l){2-11}
& Consensus & 2,197 & 1,544 & 4,936 & 791 &
423 & 417 & 558 & 4,223 & 1,885 \\
\bottomrule
\end{tabular}
\end{table}
\end{landscape}
\pagestyle{scrheadings}


\subsection{Curated sets}\label{subsec:Trans_curatedSets}
Protein coding genes that were characterised consistently
as any of the previous categories across the five datasets
are provided as supplementary data.


\section{Discussion}\label{sec:Trans_discussion}
\TK{when I started no paper, now overloads}: list them and compare to my own
work. Meta-analysis\ldots

Small overlap of tissues: so scientifically not very good.
However, most of the genes expressed everywhere.
But, in general Biology >>> Technical.

Apart Castle but Castle method not really used any more. So it might be because of
the protocol which is harder to set up or not mature (maybe can be improved) or
maybe the method gives different measurements due to what it fishes and
normalisation methods which are not very good for that (biases).

Quite expectedly, as these analyses were incorporating more studies,
the results improved greatly.
Indeed, as I focalise the study on the shared genes across the studies,
I bias the analyses towards the genes with more robust expression profiles
that may be measured with \Rnaseq.



Petit blabla sur la normalisation (de nouveau), normalisation pas adapté
ERCC ptet 1 idée mais pas vraiment un internal standard.


New set of tissue spe.
Hampel method
Quantile normalisation\ldots

MAJ tissues spe de Tiger (ne pas oublier la discussion sur l'annotation)

tend vers une 1 limite mais pas mal de redéfinition

\TK{start speaking about the normalisation a bit, deseq, tpm, \ldots}

Ouverture: là, où au début on avait pas beaucoup de data avec des réplicats
biologiques pour le normal, on a commencé en avoir pas mal. Du coup, \nuno\
a analysé les genes avec un faible coeff de var. On a défini ca comme housekeeping
genes.







