\chapter{Consistency through different transcriptomics studies}
\label{ch:Transcriptomics}
\begin{comment}
As previously stated in the introduction, while I started this project,
there wasn't any study that was investigating in-depth the reproducibility of
transcriptomics.

However, in August 2014 a paper comparing different preparation methods,
sequencing technologies operated in different laboratories
on samples from the same biological source concludes that while absolute
measurements are not consistent, the relative expression is highly consistent
(95\% Pearson correlation?)


Example:~\cite{schwanhausserglobal:2011}
\end{comment}

\section{introduction}

Why?

\begin{itemize}
    \item Difficulty of sampling of normal conditions for human, particularly
        for the solid tissues. (could be very useful in case of cancer for
        example)
    \item Lots of transcriptomics data in the repository of EBI. Can we use
        this as a base for reference?
    \item previous point not new, but recently multiplications of studies on
        normal human studies based on \Rnaseq
    \item Indeed, in the past people tried with microarray data but didn't work
        \TK{find sources}
    \item When I started this study, there wasn't any publication on this matter
        yet, but \Rnaseq was considered as quantitative (when microarrays were
        considered semi-quantitative at best).
    \item nobody did published on my specific subject yet, but since then things changed
    \item first paper from MACQ/SEC III
    \item increasing number of papers that been published on the subject
    \item I will discuss the different part of the papers that are relevant
        with additions on what my study confirms/refutes or expands them.
\end{itemize}

\section{Available studies}
\label{sec:Trans_AvailableStudies}

I chose with datasets (more precisely their samples) that were fitting the
following criteria: they had to comprise human normal samples from at least
two kind of tissues, which had been sequenced by \Rnaseq and that the raw data was
available.

The datasets used are described in the chronological order of their first publications.

Other datasets had also been considered, but not used in this study. Few reasons to
this; the accessibility/possible usage of the data was the main reason.

For example, \citet{Burge} dataset couldn't be used as raw data; for a few of the
libraries, it was impossible to determine which was the used encoding. There were
also few problems with some other thing.

\begin{comment}
\rough{Dataset: \begin{itemize}
        \item why?
        \item Main findings (particularly the ones that impact me)
        \item how they created it
\end{itemize}}
\end{comment}

\subsection{Castle et al. dataset}

This dataset has been published by \citet*{castleData} who were interested to explore
with sequencing-based technology the whole RNA repertoire. At that time, the common
employed amplification methods were inadequate to study non-polyadenylated
\glspl{RNA} which is the case of most of the \glspl{ncRNA}.

They designed the amplification step to avoid the \glspl{RNA}
smaller than 50 \gls{nt} in order to deplete their pools of long
\glspl{rRNA} \ribo{$28$S}, \ribo{$18$S}, \ribo{$16$S},
\ribo{$12$S}. Unfortunately, most mature \glspl{miRNA} are not caught either while
\glspl{rRNA} such as \ribo{$5.8$S} and \ribo{$5$S} are still amplified.

They found coding genes could be highly tissue-enriched, e.g. \gene{PRM$2$}
in \tissue{Testis}. They focused their study on the non coding part essentially
on \glspl{snoRNA},~\glspl{scaRNA},~\glspl{scRNA},~\glspl{tRNA} and mitochondrial
\glspl{RNA}.\\
They finally conclude that \glspl{ncRNA} have higher and more tissue-specific
expression patterns than \glspl{mRNA}.

They used multiple-donors pooled tissues samples (purchased as total \gls{RNA}
from Ambion (Austin, USA)).
They prepared the libraries following a whole transcriptomic protocol \citep{Armour:2009}:
specific oligonucleotides are used to amplify by \gls{PCR} nonribosomal \gls{RNA}
transcripts.

They generated an average of 50 millions sequence reads per tissue
using an Illumina GA-II sequencer. They trimmed their original reads (50 \gls{nt}
and 36 \gls{nt}) to 28 \gls{nt} that they released through EMBL archives (\ENA{ERP000257}
and \ArrayExpress{E-MTAB-305}).

Despite several limitations (lack of replicates, old technology, small reads),
I used this dataset for two main reasons. On one hand it is the oldest \Rnaseq\
data I found that was performed on Human normal tissues and on the other hand it
is the only publicly available dataset comprising \glspl{ncRNA}.


\subsection{Illumina Body Map 2.0}

This dataset has been created by Illumina mostly to advertise its new (at that time)
technology and the advantages of paired-end technology over single end.
\citep{illuminaBM}


The main characteristics of the different datasets are summarised
in~\cref{fig:Trans5DF}.

\begin{sidewaystable}{}
    \centering
    \caption{\label{fig:Trans5DF}Technical description of the 5 transcriptomic
    dataset (\Rnaseq)
     used for this study}
     \begin{tabular}{@{}ccccccccc@{}}
         \toprule
         \multicolumn{1}{c|}{\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}
             \gls{ArrayExpress} ID \\ or Consortium name
             \end{tabular}}} &
         \multicolumn{2}{c|}{\begin{tabular}[c]{@{}c@{}}
                Library\\ preparation
             \end{tabular}} &
         \multicolumn{2}{c|}{Sequencing} &
         \multicolumn{2}{c|}{Replicates} &
         \multicolumn{1}{c|}{\multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}
                     Tissue\\ number
                 \end{tabular}}} &
         \multirow{2}{*}{\begin{tabular}[c]{@{}c@{}}
         Multi-sampling \\ from the same\\ individual\end{tabular}} \\
         \cmidrule(lr){2-7}
         \multicolumn{1}{c|}{} & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}
             Whole\\  RNA\end{tabular}} &
             \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}
                 PolyA\\ selected\end{tabular}} &
                 \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}
                     Single\\ end\end{tabular}} &
                     \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}
                     Paired\\ end\end{tabular}} & \multicolumn{1}{c|}{Biological} &
                     \multicolumn{1}{c|}{Technical} &
                     \multicolumn{1}{c|}{} &  \\ \midrule
             E-MTAB-305 & Y &  & Y &  &  &  & 11 &  \\
             E-GEOD-303522 (VT) &  & Y & Y &  & Y &  & 8 &  \\
             \begin{tabular}[c]{@{}c@{}}E-MTAB-513, (IBM\: Illumina\\
             Body Map)\end{tabular} &  & Y & Y & Y &  & (Y) & 16 &  \\
                 E-MTAB-28363 &  & Y &  & Y & Y & Y & 27 &  \\
                 GTEX (v1.4) 4 &  & Y &  & Y & Y &  & 54 & Y \\ \bottomrule
    \end{tabular}
\end{sidewaystable}


\section{Consistency of processing methodology}\label{sec:Trans_consistentMethodo}

    \subsection{Reuse of processed data issues}\label{subsec:Trans_reuseOfData}

    \subsection{Genome build and annotation impact}\label{subsec:Trans_AnnotImpact}

\section{Results}\label{sec:Trans_Results}

    \subsection{Reproducibility of expression profile at tissue level}\label{subsec:Trans_ReproExpresTissue}

        \subsubsection{Correlation}\label{subsubsec:Trans_Tissue_Corr}
        \subsubsection{Clustering}\label{subsubsec:Trans_Tissue_cluster}

    \subsection{Reproducibility of expression profile at gene level}\label{subsec:Trans_ReproExpresGene}

    \subsection{Tissue specific,housekeeping genes and other categories}\label{subsec:Trans_TissueSpeAndHK}

    \subsection{Curated sets}\label{subsec:Trans_curatedSets}

\section{Discussion}\label{sec:Trans_discussion}



